Start MongoDB.
2018-07-15T19:36:51.018-0400 I CONTROL  [initandlisten] MongoDB starting : pid=12523 port=27017 dbpath=/data/db 64-bit host=mm-mac-7505.local
2018-07-15T19:36:51.019-0400 I CONTROL  [initandlisten] db version v3.6.5
2018-07-15T19:36:51.019-0400 I CONTROL  [initandlisten] git version: a20ecd3e3a174162052ff99913bc2ca9a839d618
2018-07-15T19:36:51.019-0400 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018
2018-07-15T19:36:51.019-0400 I CONTROL  [initandlisten] allocator: system
2018-07-15T19:36:51.019-0400 I CONTROL  [initandlisten] modules: none
2018-07-15T19:36:51.019-0400 I CONTROL  [initandlisten] build environment:
2018-07-15T19:36:51.019-0400 I CONTROL  [initandlisten]     distarch: x86_64
2018-07-15T19:36:51.019-0400 I CONTROL  [initandlisten]     target_arch: x86_64
2018-07-15T19:36:51.019-0400 I CONTROL  [initandlisten] options: {}
2018-07-15T19:36:51.020-0400 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2018-07-15T19:36:51.020-0400 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7680M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),cache_cursors=false,log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2018-07-15T19:36:51.903-0400 I STORAGE  [initandlisten] WiredTiger message [1531697811:903932][12523:0x7fffbd8503c0], txn-recover: Main recovery loop: starting at 52/4864
2018-07-15T19:36:52.027-0400 I STORAGE  [initandlisten] WiredTiger message [1531697812:27761][12523:0x7fffbd8503c0], txn-recover: Recovering log 52 through 53
2018-07-15T19:36:52.122-0400 I STORAGE  [initandlisten] WiredTiger message [1531697812:122294][12523:0x7fffbd8503c0], txn-recover: Recovering log 53 through 53
2018-07-15T19:36:52.193-0400 I STORAGE  [initandlisten] WiredTiger message [1531697812:193068][12523:0x7fffbd8503c0], txn-recover: Set global recovery timestamp: 0
2018-07-15T19:36:52.324-0400 I CONTROL  [initandlisten] 
2018-07-15T19:36:52.324-0400 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2018-07-15T19:36:52.324-0400 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2018-07-15T19:36:52.324-0400 I CONTROL  [initandlisten] 
2018-07-15T19:36:52.324-0400 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2018-07-15T19:36:52.324-0400 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2018-07-15T19:36:52.324-0400 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2018-07-15T19:36:52.324-0400 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2018-07-15T19:36:52.324-0400 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2018-07-15T19:36:52.325-0400 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2018-07-15T19:36:52.325-0400 I CONTROL  [initandlisten] 
2018-07-15T19:36:52.325-0400 I CONTROL  [initandlisten] 
2018-07-15T19:36:52.325-0400 I CONTROL  [initandlisten] ** WARNING: soft rlimits too low. Number of files is 256, should be at least 1000
2018-07-15T19:36:52.347-0400 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2018-07-15T19:36:52.349-0400 I NETWORK  [initandlisten] waiting for connections on port 27017
2018-07-15T19:37:23.883-0400 I CONTROL  [signalProcessingThread] got signal 15 (Terminated: 15), will terminate after current cmd ends
2018-07-15T19:37:23.883-0400 I NETWORK  [signalProcessingThread] shutdown: going to close listening sockets...
2018-07-15T19:37:23.883-0400 I NETWORK  [signalProcessingThread] removing socket file: /tmp/mongodb-27017.sock
2018-07-15T19:37:23.884-0400 I FTDC     [signalProcessingThread] Shutting down full-time diagnostic data capture
2018-07-15T19:37:23.885-0400 I STORAGE  [signalProcessingThread] WiredTigerKVEngine shutting down
2018-07-15T19:37:24.051-0400 I STORAGE  [signalProcessingThread] shutdown: removing fs lock...
2018-07-15T19:37:24.052-0400 I CONTROL  [signalProcessingThread] now exiting
2018-07-15T19:37:24.052-0400 I CONTROL  [signalProcessingThread] shutting down with code:0
Start MongoDB.
2018-07-15T20:36:45.887-0400 I CONTROL  [initandlisten] MongoDB starting : pid=13805 port=27017 dbpath=/data/db 64-bit host=mm-mac-7505.local
2018-07-15T20:36:45.888-0400 I CONTROL  [initandlisten] db version v3.6.5
2018-07-15T20:36:45.888-0400 I CONTROL  [initandlisten] git version: a20ecd3e3a174162052ff99913bc2ca9a839d618
2018-07-15T20:36:45.888-0400 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018
2018-07-15T20:36:45.888-0400 I CONTROL  [initandlisten] allocator: system
2018-07-15T20:36:45.888-0400 I CONTROL  [initandlisten] modules: none
2018-07-15T20:36:45.888-0400 I CONTROL  [initandlisten] build environment:
2018-07-15T20:36:45.888-0400 I CONTROL  [initandlisten]     distarch: x86_64
2018-07-15T20:36:45.888-0400 I CONTROL  [initandlisten]     target_arch: x86_64
2018-07-15T20:36:45.888-0400 I CONTROL  [initandlisten] options: {}
2018-07-15T20:36:45.889-0400 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2018-07-15T20:36:45.889-0400 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7680M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),cache_cursors=false,log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2018-07-15T20:36:46.785-0400 I STORAGE  [initandlisten] WiredTiger message [1531701406:785901][13805:0x7fffbd8503c0], txn-recover: Main recovery loop: starting at 53/4864
2018-07-15T20:36:46.902-0400 I STORAGE  [initandlisten] WiredTiger message [1531701406:902363][13805:0x7fffbd8503c0], txn-recover: Recovering log 53 through 54
2018-07-15T20:36:46.982-0400 I STORAGE  [initandlisten] WiredTiger message [1531701406:982015][13805:0x7fffbd8503c0], txn-recover: Recovering log 54 through 54
2018-07-15T20:36:47.041-0400 I STORAGE  [initandlisten] WiredTiger message [1531701407:41851][13805:0x7fffbd8503c0], txn-recover: Set global recovery timestamp: 0
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] 
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] 
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] 
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] 
2018-07-15T20:36:47.175-0400 I CONTROL  [initandlisten] ** WARNING: soft rlimits too low. Number of files is 256, should be at least 1000
2018-07-15T20:36:47.194-0400 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2018-07-15T20:36:47.194-0400 I NETWORK  [initandlisten] waiting for connections on port 27017
MongoDB is already running.
2018-07-15T20:37:00.362-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60289 #1 (1 connection now open)
2018-07-15T20:37:00.364-0400 I NETWORK  [conn1] received client metadata from 127.0.0.1:60289 conn1: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T20:37:00.875-0400 I NETWORK  [conn1] end connection 127.0.0.1:60289 (0 connections now open)
MongoDB is already running.
2018-07-15T20:37:17.411-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60294 #2 (1 connection now open)
2018-07-15T20:37:17.411-0400 I NETWORK  [conn2] end connection 127.0.0.1:60294 (0 connections now open)
MongoDB is already running.
2018-07-15T21:00:12.699-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60561 #3 (1 connection now open)
2018-07-15T21:00:12.699-0400 I NETWORK  [conn3] end connection 127.0.0.1:60561 (0 connections now open)
MongoDB is already running.
2018-07-15T21:01:03.169-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60569 #4 (1 connection now open)
2018-07-15T21:01:03.170-0400 I NETWORK  [conn4] end connection 127.0.0.1:60569 (0 connections now open)
MongoDB is already running.
2018-07-15T21:01:22.253-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60575 #5 (1 connection now open)
2018-07-15T21:01:22.256-0400 I NETWORK  [conn5] received client metadata from 127.0.0.1:60575 conn5: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T21:01:22.768-0400 I NETWORK  [conn5] end connection 127.0.0.1:60575 (0 connections now open)
MongoDB is already running.
2018-07-15T21:02:34.864-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60585 #6 (1 connection now open)
2018-07-15T21:02:34.865-0400 I NETWORK  [conn6] received client metadata from 127.0.0.1:60585 conn6: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T21:02:34.871-0400 I NETWORK  [conn6] end connection 127.0.0.1:60585 (0 connections now open)
MongoDB is already running.
2018-07-15T21:02:55.081-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60590 #7 (1 connection now open)
2018-07-15T21:02:55.081-0400 I NETWORK  [conn7] received client metadata from 127.0.0.1:60590 conn7: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T21:02:55.087-0400 I NETWORK  [conn7] end connection 127.0.0.1:60590 (0 connections now open)
MongoDB is already running.
2018-07-15T21:04:32.758-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60601 #8 (1 connection now open)
2018-07-15T21:04:32.759-0400 I NETWORK  [conn8] received client metadata from 127.0.0.1:60601 conn8: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T21:04:33.273-0400 I NETWORK  [conn8] end connection 127.0.0.1:60601 (0 connections now open)
MongoDB is already running.
2018-07-15T21:05:07.182-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60609 #9 (1 connection now open)
2018-07-15T21:05:07.182-0400 I NETWORK  [conn9] received client metadata from 127.0.0.1:60609 conn9: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T21:05:07.697-0400 I NETWORK  [conn9] end connection 127.0.0.1:60609 (0 connections now open)
MongoDB is already running.
2018-07-15T21:15:26.819-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61129 #10 (1 connection now open)
2018-07-15T21:15:26.819-0400 I NETWORK  [conn10] end connection 127.0.0.1:61129 (0 connections now open)
MongoDB is already running.
2018-07-15T21:16:16.174-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61138 #11 (1 connection now open)
2018-07-15T21:16:16.174-0400 I NETWORK  [conn11] received client metadata from 127.0.0.1:61138 conn11: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T21:16:16.181-0400 I NETWORK  [conn11] end connection 127.0.0.1:61138 (0 connections now open)
MongoDB is already running.
2018-07-15T21:17:33.872-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61148 #12 (1 connection now open)
2018-07-15T21:17:33.873-0400 I NETWORK  [conn12] received client metadata from 127.0.0.1:61148 conn12: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T21:17:33.879-0400 I NETWORK  [conn12] end connection 127.0.0.1:61148 (0 connections now open)
MongoDB is already running.
2018-07-15T21:39:27.274-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61397 #13 (1 connection now open)
2018-07-15T21:39:27.275-0400 I NETWORK  [conn13] received client metadata from 127.0.0.1:61397 conn13: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T21:39:27.788-0400 I NETWORK  [conn13] end connection 127.0.0.1:61397 (0 connections now open)
MongoDB is already running.
2018-07-15T21:40:22.437-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61404 #14 (1 connection now open)
2018-07-15T21:40:22.437-0400 I NETWORK  [conn14] end connection 127.0.0.1:61404 (0 connections now open)
MongoDB is already running.
2018-07-15T21:44:09.239-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61599 #15 (1 connection now open)
2018-07-15T21:44:09.240-0400 I NETWORK  [conn15] received client metadata from 127.0.0.1:61599 conn15: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T21:44:09.240-0400 I NETWORK  [conn15] end connection 127.0.0.1:61599 (0 connections now open)
MongoDB is already running.
2018-07-15T21:45:17.338-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61606 #16 (1 connection now open)
2018-07-15T21:45:17.338-0400 I NETWORK  [conn16] end connection 127.0.0.1:61606 (0 connections now open)
MongoDB is already running.
2018-07-15T21:47:21.086-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61618 #17 (1 connection now open)
2018-07-15T21:47:21.087-0400 I NETWORK  [conn17] end connection 127.0.0.1:61618 (0 connections now open)
MongoDB is already running.
2018-07-15T21:47:49.342-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61625 #18 (1 connection now open)
2018-07-15T21:47:49.342-0400 I NETWORK  [conn18] end connection 127.0.0.1:61625 (0 connections now open)
MongoDB is already running.
2018-07-15T21:50:05.116-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61640 #19 (1 connection now open)
2018-07-15T21:50:05.116-0400 I NETWORK  [conn19] end connection 127.0.0.1:61640 (0 connections now open)
MongoDB is already running.
2018-07-15T21:50:56.094-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61647 #20 (1 connection now open)
2018-07-15T21:50:56.094-0400 I NETWORK  [conn20] end connection 127.0.0.1:61647 (0 connections now open)
MongoDB is already running.
2018-07-15T21:56:16.837-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61672 #21 (1 connection now open)
2018-07-15T21:56:16.837-0400 I NETWORK  [conn21] received client metadata from 127.0.0.1:61672 conn21: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T21:56:16.838-0400 I NETWORK  [conn21] end connection 127.0.0.1:61672 (0 connections now open)
MongoDB is already running.
2018-07-15T22:00:21.049-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61690 #22 (1 connection now open)
2018-07-15T22:00:21.050-0400 I NETWORK  [conn22] received client metadata from 127.0.0.1:61690 conn22: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:00:21.555-0400 I NETWORK  [conn22] end connection 127.0.0.1:61690 (0 connections now open)
MongoDB is already running.
2018-07-15T22:00:35.659-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61695 #23 (1 connection now open)
2018-07-15T22:00:35.659-0400 I NETWORK  [conn23] received client metadata from 127.0.0.1:61695 conn23: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:00:36.173-0400 I NETWORK  [conn23] end connection 127.0.0.1:61695 (0 connections now open)
MongoDB is already running.
2018-07-15T22:01:04.035-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61700 #24 (1 connection now open)
2018-07-15T22:01:04.036-0400 I NETWORK  [conn24] end connection 127.0.0.1:61700 (0 connections now open)
MongoDB is already running.
2018-07-15T22:01:51.457-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61708 #25 (1 connection now open)
2018-07-15T22:01:51.457-0400 I NETWORK  [conn25] received client metadata from 127.0.0.1:61708 conn25: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:01:51.459-0400 I NETWORK  [conn25] end connection 127.0.0.1:61708 (0 connections now open)
MongoDB is already running.
2018-07-15T22:02:39.324-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61716 #26 (1 connection now open)
2018-07-15T22:02:39.324-0400 I NETWORK  [conn26] received client metadata from 127.0.0.1:61716 conn26: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:02:39.325-0400 I NETWORK  [conn26] end connection 127.0.0.1:61716 (0 connections now open)
MongoDB is already running.
2018-07-15T22:07:05.903-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61777 #27 (1 connection now open)
2018-07-15T22:07:05.905-0400 I NETWORK  [conn27] received client metadata from 127.0.0.1:61777 conn27: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:07:05.943-0400 I NETWORK  [conn27] end connection 127.0.0.1:61777 (0 connections now open)
MongoDB is already running.
2018-07-15T22:12:43.545-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61929 #28 (1 connection now open)
2018-07-15T22:12:43.546-0400 I NETWORK  [conn28] received client metadata from 127.0.0.1:61929 conn28: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:12:43.585-0400 I NETWORK  [conn28] end connection 127.0.0.1:61929 (0 connections now open)
MongoDB is already running.
2018-07-15T22:14:11.713-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61946 #29 (1 connection now open)
2018-07-15T22:14:11.714-0400 I NETWORK  [conn29] received client metadata from 127.0.0.1:61946 conn29: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:14:12.222-0400 I NETWORK  [conn29] end connection 127.0.0.1:61946 (0 connections now open)
MongoDB is already running.
2018-07-15T22:17:43.844-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62081 #30 (1 connection now open)
2018-07-15T22:17:43.844-0400 I NETWORK  [conn30] received client metadata from 127.0.0.1:62081 conn30: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:17:43.878-0400 I NETWORK  [conn30] end connection 127.0.0.1:62081 (0 connections now open)
MongoDB is already running.
2018-07-15T22:30:10.344-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62148 #31 (1 connection now open)
2018-07-15T22:30:10.344-0400 I NETWORK  [conn31] received client metadata from 127.0.0.1:62148 conn31: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:30:10.350-0400 I NETWORK  [conn31] end connection 127.0.0.1:62148 (0 connections now open)
MongoDB is already running.
2018-07-15T22:30:47.862-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62159 #32 (1 connection now open)
2018-07-15T22:30:47.863-0400 I NETWORK  [conn32] received client metadata from 127.0.0.1:62159 conn32: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:30:47.869-0400 I NETWORK  [conn32] end connection 127.0.0.1:62159 (0 connections now open)
2018-07-15T22:41:13.110-0400 I CONTROL  [signalProcessingThread] got signal 15 (Terminated: 15), will terminate after current cmd ends
2018-07-15T22:41:13.110-0400 I NETWORK  [signalProcessingThread] shutdown: going to close listening sockets...
2018-07-15T22:41:13.110-0400 I NETWORK  [signalProcessingThread] removing socket file: /tmp/mongodb-27017.sock
2018-07-15T22:41:13.110-0400 I FTDC     [signalProcessingThread] Shutting down full-time diagnostic data capture
2018-07-15T22:41:13.114-0400 I STORAGE  [signalProcessingThread] WiredTigerKVEngine shutting down
2018-07-15T22:41:13.333-0400 I STORAGE  [signalProcessingThread] shutdown: removing fs lock...
2018-07-15T22:41:13.334-0400 I CONTROL  [signalProcessingThread] now exiting
2018-07-15T22:41:13.334-0400 I CONTROL  [signalProcessingThread] shutting down with code:0
Start MongoDB.
2018-07-15T22:44:45.520-0400 I CONTROL  [initandlisten] MongoDB starting : pid=16728 port=27017 dbpath=/data/db 64-bit host=mm-mac-7505.local
2018-07-15T22:44:45.520-0400 I CONTROL  [initandlisten] db version v3.6.5
2018-07-15T22:44:45.520-0400 I CONTROL  [initandlisten] git version: a20ecd3e3a174162052ff99913bc2ca9a839d618
2018-07-15T22:44:45.520-0400 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018
2018-07-15T22:44:45.520-0400 I CONTROL  [initandlisten] allocator: system
2018-07-15T22:44:45.520-0400 I CONTROL  [initandlisten] modules: none
2018-07-15T22:44:45.520-0400 I CONTROL  [initandlisten] build environment:
2018-07-15T22:44:45.520-0400 I CONTROL  [initandlisten]     distarch: x86_64
2018-07-15T22:44:45.520-0400 I CONTROL  [initandlisten]     target_arch: x86_64
2018-07-15T22:44:45.520-0400 I CONTROL  [initandlisten] options: {}
2018-07-15T22:44:45.521-0400 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2018-07-15T22:44:45.521-0400 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7680M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),cache_cursors=false,log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2018-07-15T22:44:46.463-0400 I STORAGE  [initandlisten] WiredTiger message [1531709086:463819][16728:0x7fffbd8503c0], txn-recover: Main recovery loop: starting at 54/22656
2018-07-15T22:44:46.579-0400 I STORAGE  [initandlisten] WiredTiger message [1531709086:579430][16728:0x7fffbd8503c0], txn-recover: Recovering log 54 through 55
2018-07-15T22:44:46.669-0400 I STORAGE  [initandlisten] WiredTiger message [1531709086:669971][16728:0x7fffbd8503c0], txn-recover: Recovering log 55 through 55
2018-07-15T22:44:46.732-0400 I STORAGE  [initandlisten] WiredTiger message [1531709086:732556][16728:0x7fffbd8503c0], txn-recover: Set global recovery timestamp: 0
2018-07-15T22:44:46.903-0400 I CONTROL  [initandlisten] 
2018-07-15T22:44:46.903-0400 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2018-07-15T22:44:46.903-0400 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2018-07-15T22:44:46.903-0400 I CONTROL  [initandlisten] 
2018-07-15T22:44:46.903-0400 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2018-07-15T22:44:46.903-0400 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2018-07-15T22:44:46.903-0400 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2018-07-15T22:44:46.903-0400 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2018-07-15T22:44:46.903-0400 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2018-07-15T22:44:46.903-0400 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2018-07-15T22:44:46.903-0400 I CONTROL  [initandlisten] 
2018-07-15T22:44:46.921-0400 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2018-07-15T22:44:46.923-0400 I NETWORK  [initandlisten] waiting for connections on port 27017
MongoDB is already running.
2018-07-15T22:45:03.742-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62573 #1 (1 connection now open)
2018-07-15T22:45:03.743-0400 I NETWORK  [conn1] received client metadata from 127.0.0.1:62573 conn1: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:45:03.752-0400 I NETWORK  [conn1] end connection 127.0.0.1:62573 (0 connections now open)
MongoDB is already running.
2018-07-15T22:45:15.682-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62582 #2 (1 connection now open)
2018-07-15T22:45:15.683-0400 I NETWORK  [conn2] received client metadata from 127.0.0.1:62582 conn2: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-15T22:45:15.688-0400 I NETWORK  [conn2] end connection 127.0.0.1:62582 (0 connections now open)
2018-07-17T01:15:58.785-0400 I COMMAND  [thread3] command config.$cmd command: createIndexes { createIndexes: "system.sessions", indexes: [ { key: { lastUse: 1 }, name: "lsidTTLIndex", expireAfterSeconds: 1800 } ], $db: "config" } numYields:0 reslen:134 locks:{ Global: { acquireCount: { r: 1, w: 1 } }, Database: { acquireCount: { W: 1 }, acquireWaitCount: { W: 1 }, timeAcquiringMicros: { W: 285846 } } } protocol:op_msg 297ms
2018-07-17T04:52:13.414-0400 I COMMAND  [PeriodicTaskRunner] task: UnusedLockCleaner took: 143ms
Start MongoDB.
2018-07-18T11:34:22.907-0400 I CONTROL  [initandlisten] MongoDB starting : pid=2288 port=27017 dbpath=/data/db 64-bit host=mm-mac-7505.local
2018-07-18T11:34:22.907-0400 I CONTROL  [initandlisten] db version v3.6.5
2018-07-18T11:34:22.907-0400 I CONTROL  [initandlisten] git version: a20ecd3e3a174162052ff99913bc2ca9a839d618
2018-07-18T11:34:22.907-0400 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018
2018-07-18T11:34:22.907-0400 I CONTROL  [initandlisten] allocator: system
2018-07-18T11:34:22.907-0400 I CONTROL  [initandlisten] modules: none
2018-07-18T11:34:22.907-0400 I CONTROL  [initandlisten] build environment:
2018-07-18T11:34:22.907-0400 I CONTROL  [initandlisten]     distarch: x86_64
2018-07-18T11:34:22.907-0400 I CONTROL  [initandlisten]     target_arch: x86_64
2018-07-18T11:34:22.907-0400 I CONTROL  [initandlisten] options: {}
2018-07-18T11:34:22.908-0400 W -        [initandlisten] Detected unclean shutdown - /data/db/mongod.lock is not empty.
2018-07-18T11:34:22.909-0400 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2018-07-18T11:34:22.909-0400 W STORAGE  [initandlisten] Recovering data from the last clean checkpoint.
2018-07-18T11:34:22.909-0400 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7680M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),cache_cursors=false,log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2018-07-18T11:34:23.863-0400 I STORAGE  [initandlisten] WiredTiger message [1531928063:863783][2288:0x7fffdb5633c0], txn-recover: Main recovery loop: starting at 55/4480
2018-07-18T11:34:23.864-0400 I STORAGE  [initandlisten] WiredTiger message [1531928063:864325][2288:0x7fffdb5633c0], txn-recover: Recovering log 55 through 56
2018-07-18T11:34:23.936-0400 I STORAGE  [initandlisten] WiredTiger message [1531928063:936377][2288:0x7fffdb5633c0], txn-recover: Recovering log 56 through 56
2018-07-18T11:34:23.984-0400 I STORAGE  [initandlisten] WiredTiger message [1531928063:984945][2288:0x7fffdb5633c0], txn-recover: Set global recovery timestamp: 0
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] 
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] 
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] 
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] 
2018-07-18T11:34:24.157-0400 I CONTROL  [initandlisten] ** WARNING: soft rlimits too low. Number of files is 256, should be at least 1000
2018-07-18T11:34:24.216-0400 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2018-07-18T11:34:24.218-0400 I NETWORK  [initandlisten] waiting for connections on port 27017
2018-07-18T11:34:25.018-0400 I FTDC     [ftdc] Unclean full-time diagnostic data capture shutdown detected, found interim file, some metrics may have been lost. OK
MongoDB is already running.
2018-07-18T13:27:18.815-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:59658 #1 (1 connection now open)
2018-07-18T13:27:18.816-0400 I NETWORK  [conn1] received client metadata from 127.0.0.1:59658 conn1: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T13:27:18.821-0400 I NETWORK  [conn1] end connection 127.0.0.1:59658 (0 connections now open)
Stop MongoDB.
2018-07-18T13:28:37.081-0400 I CONTROL  [signalProcessingThread] got signal 15 (Terminated: 15), will terminate after current cmd ends
2018-07-18T13:28:37.081-0400 I NETWORK  [signalProcessingThread] shutdown: going to close listening sockets...
2018-07-18T13:28:37.081-0400 I NETWORK  [signalProcessingThread] removing socket file: /tmp/mongodb-27017.sock
2018-07-18T13:28:37.082-0400 I FTDC     [signalProcessingThread] Shutting down full-time diagnostic data capture
2018-07-18T13:28:37.086-0400 I STORAGE  [signalProcessingThread] WiredTigerKVEngine shutting down
2018-07-18T13:28:37.234-0400 I STORAGE  [signalProcessingThread] shutdown: removing fs lock...
2018-07-18T13:28:37.235-0400 I CONTROL  [signalProcessingThread] now exiting
2018-07-18T13:28:37.235-0400 I CONTROL  [signalProcessingThread] shutting down with code:0
Start MongoDB.
2018-07-18T14:16:27.539-0400 I CONTROL  [initandlisten] MongoDB starting : pid=5779 port=27017 dbpath=/data/db 64-bit host=mm-mac-7505.local
2018-07-18T14:16:27.540-0400 I CONTROL  [initandlisten] db version v3.6.5
2018-07-18T14:16:27.540-0400 I CONTROL  [initandlisten] git version: a20ecd3e3a174162052ff99913bc2ca9a839d618
2018-07-18T14:16:27.540-0400 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018
2018-07-18T14:16:27.540-0400 I CONTROL  [initandlisten] allocator: system
2018-07-18T14:16:27.540-0400 I CONTROL  [initandlisten] modules: none
2018-07-18T14:16:27.540-0400 I CONTROL  [initandlisten] build environment:
2018-07-18T14:16:27.540-0400 I CONTROL  [initandlisten]     distarch: x86_64
2018-07-18T14:16:27.540-0400 I CONTROL  [initandlisten]     target_arch: x86_64
2018-07-18T14:16:27.540-0400 I CONTROL  [initandlisten] options: {}
2018-07-18T14:16:27.543-0400 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2018-07-18T14:16:27.543-0400 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7680M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),cache_cursors=false,log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2018-07-18T14:16:28.423-0400 I STORAGE  [initandlisten] WiredTiger message [1531937788:423091][5779:0x7fffdb5633c0], txn-recover: Main recovery loop: starting at 56/20096
2018-07-18T14:16:28.523-0400 I STORAGE  [initandlisten] WiredTiger message [1531937788:523153][5779:0x7fffdb5633c0], txn-recover: Recovering log 56 through 57
2018-07-18T14:16:28.594-0400 I STORAGE  [initandlisten] WiredTiger message [1531937788:594260][5779:0x7fffdb5633c0], txn-recover: Recovering log 57 through 57
2018-07-18T14:16:28.647-0400 I STORAGE  [initandlisten] WiredTiger message [1531937788:647564][5779:0x7fffdb5633c0], txn-recover: Set global recovery timestamp: 0
2018-07-18T14:16:28.784-0400 I CONTROL  [initandlisten] 
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] 
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] 
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] 
2018-07-18T14:16:28.785-0400 I CONTROL  [initandlisten] ** WARNING: soft rlimits too low. Number of files is 256, should be at least 1000
2018-07-18T14:16:28.801-0400 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2018-07-18T14:16:28.803-0400 I NETWORK  [initandlisten] waiting for connections on port 27017
MongoDB is already running.
2018-07-18T14:18:41.547-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60056 #1 (1 connection now open)
2018-07-18T14:18:41.547-0400 I NETWORK  [conn1] received client metadata from 127.0.0.1:60056 conn1: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:18:42.062-0400 I NETWORK  [conn1] end connection 127.0.0.1:60056 (0 connections now open)
MongoDB is already running.
2018-07-18T14:19:18.709-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60066 #2 (1 connection now open)
2018-07-18T14:19:18.710-0400 I NETWORK  [conn2] received client metadata from 127.0.0.1:60066 conn2: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:19:19.221-0400 I NETWORK  [conn2] end connection 127.0.0.1:60066 (0 connections now open)
MongoDB is already running.
2018-07-18T14:19:40.514-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60072 #3 (1 connection now open)
2018-07-18T14:19:40.514-0400 I NETWORK  [conn3] received client metadata from 127.0.0.1:60072 conn3: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:19:41.023-0400 I NETWORK  [conn3] end connection 127.0.0.1:60072 (0 connections now open)
MongoDB is already running.
2018-07-18T14:19:57.845-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60077 #4 (1 connection now open)
2018-07-18T14:19:57.846-0400 I NETWORK  [conn4] received client metadata from 127.0.0.1:60077 conn4: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:19:57.848-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60078 #5 (2 connections now open)
2018-07-18T14:19:57.849-0400 I NETWORK  [conn5] received client metadata from 127.0.0.1:60078 conn5: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:19:57.854-0400 I STORAGE  [conn5] createCollection: adstxt.posts with generated UUID: 37048fa5-83f6-4c57-8d8a-09894915564c
2018-07-18T14:19:57.956-0400 I NETWORK  [conn5] end connection 127.0.0.1:60078 (1 connection now open)
2018-07-18T14:19:57.957-0400 I NETWORK  [conn4] end connection 127.0.0.1:60077 (0 connections now open)
2018-07-18T14:20:07.980-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60080 #6 (1 connection now open)
2018-07-18T14:20:07.980-0400 I NETWORK  [conn6] received client metadata from 127.0.0.1:60080 conn6: { application: { name: "MongoDB Shell" }, driver: { name: "MongoDB Internal Client", version: "3.6.5" }, os: { type: "Darwin", name: "Mac OS X", architecture: "x86_64", version: "16.7.0" } }
MongoDB is already running.
2018-07-18T14:24:04.506-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60116 #7 (2 connections now open)
2018-07-18T14:24:04.506-0400 I NETWORK  [conn7] received client metadata from 127.0.0.1:60116 conn7: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:24:04.508-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60117 #8 (3 connections now open)
2018-07-18T14:24:04.508-0400 I NETWORK  [conn8] received client metadata from 127.0.0.1:60117 conn8: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:24:04.600-0400 I NETWORK  [conn8] end connection 127.0.0.1:60117 (2 connections now open)
2018-07-18T14:24:04.600-0400 I NETWORK  [conn7] end connection 127.0.0.1:60116 (1 connection now open)
MongoDB is already running.
2018-07-18T14:25:15.553-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60125 #9 (2 connections now open)
2018-07-18T14:25:15.554-0400 I NETWORK  [conn9] received client metadata from 127.0.0.1:60125 conn9: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:25:15.555-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60126 #10 (3 connections now open)
2018-07-18T14:25:15.555-0400 I NETWORK  [conn10] received client metadata from 127.0.0.1:60126 conn10: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:25:15.643-0400 I NETWORK  [conn10] end connection 127.0.0.1:60126 (2 connections now open)
2018-07-18T14:25:15.644-0400 I NETWORK  [conn9] end connection 127.0.0.1:60125 (1 connection now open)
MongoDB is already running.
2018-07-18T14:26:34.240-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60138 #11 (2 connections now open)
2018-07-18T14:26:34.240-0400 I NETWORK  [conn11] received client metadata from 127.0.0.1:60138 conn11: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:26:34.242-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60139 #12 (3 connections now open)
2018-07-18T14:26:34.242-0400 I NETWORK  [conn12] received client metadata from 127.0.0.1:60139 conn12: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:26:34.256-0400 I NETWORK  [conn12] end connection 127.0.0.1:60139 (2 connections now open)
2018-07-18T14:26:34.256-0400 I NETWORK  [conn11] end connection 127.0.0.1:60138 (1 connection now open)
2018-07-18T14:29:19.120-0400 I NETWORK  [conn6] end connection 127.0.0.1:60080 (0 connections now open)
2018-07-18T14:29:30.543-0400 I CONTROL  [signalProcessingThread] got signal 15 (Terminated: 15), will terminate after current cmd ends
2018-07-18T14:29:30.543-0400 I NETWORK  [signalProcessingThread] shutdown: going to close listening sockets...
2018-07-18T14:29:30.543-0400 I NETWORK  [signalProcessingThread] removing socket file: /tmp/mongodb-27017.sock
2018-07-18T14:29:30.545-0400 I FTDC     [signalProcessingThread] Shutting down full-time diagnostic data capture
2018-07-18T14:29:30.547-0400 I STORAGE  [signalProcessingThread] WiredTigerKVEngine shutting down
2018-07-18T14:29:30.695-0400 I STORAGE  [signalProcessingThread] shutdown: removing fs lock...
2018-07-18T14:29:30.696-0400 I CONTROL  [signalProcessingThread] now exiting
2018-07-18T14:29:30.696-0400 I CONTROL  [signalProcessingThread] shutting down with code:0
Start MongoDB.
2018-07-18T14:29:35.704-0400 I CONTROL  [initandlisten] MongoDB starting : pid=6069 port=27017 dbpath=/data/db 64-bit host=mm-mac-7505.local
2018-07-18T14:29:35.705-0400 I CONTROL  [initandlisten] db version v3.6.5
2018-07-18T14:29:35.705-0400 I CONTROL  [initandlisten] git version: a20ecd3e3a174162052ff99913bc2ca9a839d618
2018-07-18T14:29:35.705-0400 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018
2018-07-18T14:29:35.705-0400 I CONTROL  [initandlisten] allocator: system
2018-07-18T14:29:35.705-0400 I CONTROL  [initandlisten] modules: none
2018-07-18T14:29:35.705-0400 I CONTROL  [initandlisten] build environment:
2018-07-18T14:29:35.705-0400 I CONTROL  [initandlisten]     distarch: x86_64
2018-07-18T14:29:35.705-0400 I CONTROL  [initandlisten]     target_arch: x86_64
2018-07-18T14:29:35.705-0400 I CONTROL  [initandlisten] options: {}
2018-07-18T14:29:35.705-0400 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2018-07-18T14:29:35.706-0400 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7680M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),cache_cursors=false,log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2018-07-18T14:29:36.572-0400 I STORAGE  [initandlisten] WiredTiger message [1531938576:572380][6069:0x7fffdb5633c0], txn-recover: Main recovery loop: starting at 57/116864
2018-07-18T14:29:36.681-0400 I STORAGE  [initandlisten] WiredTiger message [1531938576:681635][6069:0x7fffdb5633c0], txn-recover: Recovering log 57 through 58
2018-07-18T14:29:36.760-0400 I STORAGE  [initandlisten] WiredTiger message [1531938576:760709][6069:0x7fffdb5633c0], txn-recover: Recovering log 58 through 58
2018-07-18T14:29:36.817-0400 I STORAGE  [initandlisten] WiredTiger message [1531938576:817960][6069:0x7fffdb5633c0], txn-recover: Set global recovery timestamp: 0
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] 
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] 
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] 
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] 
2018-07-18T14:29:36.951-0400 I CONTROL  [initandlisten] ** WARNING: soft rlimits too low. Number of files is 256, should be at least 1000
2018-07-18T14:29:36.969-0400 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2018-07-18T14:29:36.970-0400 I NETWORK  [initandlisten] waiting for connections on port 27017
2018-07-18T14:29:37.275-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60185 #1 (1 connection now open)
2018-07-18T14:29:37.276-0400 I NETWORK  [conn1] received client metadata from 127.0.0.1:60185 conn1: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:29:37.277-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60186 #2 (2 connections now open)
2018-07-18T14:29:37.278-0400 I NETWORK  [conn2] received client metadata from 127.0.0.1:60186 conn2: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:29:37.295-0400 I NETWORK  [conn2] end connection 127.0.0.1:60186 (1 connection now open)
2018-07-18T14:29:37.787-0400 I NETWORK  [conn1] end connection 127.0.0.1:60185 (0 connections now open)
2018-07-18T14:29:42.313-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60188 #3 (1 connection now open)
2018-07-18T14:29:42.313-0400 I NETWORK  [conn3] received client metadata from 127.0.0.1:60188 conn3: { application: { name: "MongoDB Shell" }, driver: { name: "MongoDB Internal Client", version: "3.6.5" }, os: { type: "Darwin", name: "Mac OS X", architecture: "x86_64", version: "16.7.0" } }
2018-07-18T14:31:21.899-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T14:31:21.899-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T14:31:21.925-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T14:31:27.455-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60251 #4 (2 connections now open)
2018-07-18T14:31:27.455-0400 I NETWORK  [conn4] received client metadata from 127.0.0.1:60251 conn4: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:31:27.456-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60252 #5 (3 connections now open)
2018-07-18T14:31:27.456-0400 I NETWORK  [conn5] received client metadata from 127.0.0.1:60252 conn5: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:31:27.462-0400 I STORAGE  [conn5] createCollection: adstxt.posts with generated UUID: ab6add0a-5029-4e60-a9b1-e512d63fdbdc
2018-07-18T14:31:27.522-0400 I NETWORK  [conn4] end connection 127.0.0.1:60251 (1 connection now open)
2018-07-18T14:31:27.522-0400 I NETWORK  [conn5] end connection 127.0.0.1:60252 (2 connections now open)
2018-07-18T14:33:31.140-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T14:33:31.140-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T14:33:31.170-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T14:33:38.599-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60290 #6 (2 connections now open)
2018-07-18T14:33:38.599-0400 I NETWORK  [conn6] received client metadata from 127.0.0.1:60290 conn6: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:33:39.113-0400 I NETWORK  [conn6] end connection 127.0.0.1:60290 (1 connection now open)
MongoDB is already running.
2018-07-18T14:34:16.876-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60297 #7 (2 connections now open)
2018-07-18T14:34:16.876-0400 I NETWORK  [conn7] received client metadata from 127.0.0.1:60297 conn7: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:34:16.877-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60298 #8 (3 connections now open)
2018-07-18T14:34:16.877-0400 I NETWORK  [conn8] received client metadata from 127.0.0.1:60298 conn8: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:34:16.878-0400 I STORAGE  [conn8] createCollection: adstxt.posts with generated UUID: e0bd9759-7b34-4894-aa9c-a6f4b6949fea
2018-07-18T14:34:16.945-0400 I NETWORK  [conn8] end connection 127.0.0.1:60298 (2 connections now open)
2018-07-18T14:34:16.945-0400 I NETWORK  [conn7] end connection 127.0.0.1:60297 (1 connection now open)
2018-07-18T14:34:36.067-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T14:34:36.067-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T14:34:36.095-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T14:34:57.993-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60309 #9 (2 connections now open)
2018-07-18T14:34:57.993-0400 I NETWORK  [conn9] received client metadata from 127.0.0.1:60309 conn9: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:34:57.994-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60310 #10 (3 connections now open)
2018-07-18T14:34:57.995-0400 I NETWORK  [conn10] received client metadata from 127.0.0.1:60310 conn10: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:34:57.995-0400 I STORAGE  [conn10] createCollection: adstxt.posts with generated UUID: ea8e7a03-65aa-445c-b8c2-193dcb2376b0
2018-07-18T14:35:02.910-0400 I NETWORK  [conn10] end connection 127.0.0.1:60310 (2 connections now open)
2018-07-18T14:35:02.910-0400 I NETWORK  [conn9] end connection 127.0.0.1:60309 (1 connection now open)
2018-07-18T14:37:27.471-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T14:37:27.471-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T14:37:27.495-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T14:38:45.201-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60363 #11 (2 connections now open)
2018-07-18T14:38:45.202-0400 I NETWORK  [conn11] received client metadata from 127.0.0.1:60363 conn11: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:38:45.203-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60364 #12 (3 connections now open)
2018-07-18T14:38:45.204-0400 I NETWORK  [conn12] received client metadata from 127.0.0.1:60364 conn12: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:38:45.204-0400 I STORAGE  [conn12] createCollection: adstxt.posts with generated UUID: a1a61d91-0ef3-4cce-bf31-7445cc14afd3
2018-07-18T14:38:46.821-0400 I NETWORK  [conn12] end connection 127.0.0.1:60364 (2 connections now open)
2018-07-18T14:38:46.821-0400 I NETWORK  [conn11] end connection 127.0.0.1:60363 (1 connection now open)
MongoDB is already running.
2018-07-18T14:39:20.950-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60376 #13 (2 connections now open)
2018-07-18T14:39:20.950-0400 I NETWORK  [conn13] received client metadata from 127.0.0.1:60376 conn13: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:39:20.951-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60377 #14 (3 connections now open)
2018-07-18T14:39:20.952-0400 I NETWORK  [conn14] received client metadata from 127.0.0.1:60377 conn14: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:39:22.638-0400 I NETWORK  [conn14] end connection 127.0.0.1:60377 (2 connections now open)
2018-07-18T14:39:22.638-0400 I NETWORK  [conn13] end connection 127.0.0.1:60376 (1 connection now open)
2018-07-18T14:43:09.132-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T14:43:09.132-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T14:43:09.157-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T14:43:16.490-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60405 #15 (2 connections now open)
2018-07-18T14:43:16.490-0400 I NETWORK  [conn15] received client metadata from 127.0.0.1:60405 conn15: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:43:16.491-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60406 #16 (3 connections now open)
2018-07-18T14:43:16.491-0400 I NETWORK  [conn16] received client metadata from 127.0.0.1:60406 conn16: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:43:16.492-0400 I STORAGE  [conn16] createCollection: adstxt.posts with generated UUID: 84f5034e-4dd3-417b-b386-2d0412af549f
2018-07-18T14:43:18.036-0400 I NETWORK  [conn15] end connection 127.0.0.1:60405 (1 connection now open)
2018-07-18T14:43:18.036-0400 I NETWORK  [conn16] end connection 127.0.0.1:60406 (2 connections now open)
MongoDB is already running.
2018-07-18T14:44:28.736-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60422 #17 (2 connections now open)
2018-07-18T14:44:28.736-0400 I NETWORK  [conn17] received client metadata from 127.0.0.1:60422 conn17: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:44:28.737-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60423 #18 (3 connections now open)
2018-07-18T14:44:28.737-0400 I NETWORK  [conn18] received client metadata from 127.0.0.1:60423 conn18: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T14:44:30.214-0400 I NETWORK  [conn18] end connection 127.0.0.1:60423 (2 connections now open)
2018-07-18T14:44:30.214-0400 I NETWORK  [conn17] end connection 127.0.0.1:60422 (1 connection now open)
MongoDB is already running.
2018-07-18T15:00:48.923-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60506 #19 (2 connections now open)
2018-07-18T15:00:48.923-0400 I NETWORK  [conn19] received client metadata from 127.0.0.1:60506 conn19: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:00:48.924-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60507 #20 (3 connections now open)
2018-07-18T15:00:48.925-0400 I NETWORK  [conn20] received client metadata from 127.0.0.1:60507 conn20: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:00:50.585-0400 I NETWORK  [conn20] end connection 127.0.0.1:60507 (2 connections now open)
2018-07-18T15:00:50.585-0400 I NETWORK  [conn19] end connection 127.0.0.1:60506 (1 connection now open)
MongoDB is already running.
2018-07-18T15:04:36.771-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60538 #21 (2 connections now open)
2018-07-18T15:04:36.771-0400 I NETWORK  [conn21] received client metadata from 127.0.0.1:60538 conn21: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:04:37.280-0400 I NETWORK  [conn21] end connection 127.0.0.1:60538 (1 connection now open)
MongoDB is already running.
2018-07-18T15:05:29.408-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60551 #22 (2 connections now open)
2018-07-18T15:05:29.408-0400 I NETWORK  [conn22] received client metadata from 127.0.0.1:60551 conn22: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:05:29.409-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60552 #23 (3 connections now open)
2018-07-18T15:05:29.409-0400 I NETWORK  [conn23] received client metadata from 127.0.0.1:60552 conn23: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:05:30.943-0400 I NETWORK  [conn23] end connection 127.0.0.1:60552 (2 connections now open)
2018-07-18T15:05:30.943-0400 I NETWORK  [conn22] end connection 127.0.0.1:60551 (1 connection now open)
2018-07-18T15:06:20.064-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T15:06:20.064-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T15:06:20.089-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T15:06:46.165-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60567 #24 (2 connections now open)
2018-07-18T15:06:46.165-0400 I NETWORK  [conn24] received client metadata from 127.0.0.1:60567 conn24: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:06:46.166-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60568 #25 (3 connections now open)
2018-07-18T15:06:46.167-0400 I NETWORK  [conn25] received client metadata from 127.0.0.1:60568 conn25: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:06:46.167-0400 I STORAGE  [conn25] createCollection: adstxt.posts with generated UUID: a17fe8dd-bd2b-42cf-baba-0489ad8d2861
2018-07-18T15:06:48.009-0400 I NETWORK  [conn24] end connection 127.0.0.1:60567 (1 connection now open)
2018-07-18T15:06:48.009-0400 I NETWORK  [conn25] end connection 127.0.0.1:60568 (2 connections now open)
2018-07-18T15:07:52.172-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T15:07:52.172-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T15:07:52.197-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T15:09:42.578-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60607 #26 (2 connections now open)
2018-07-18T15:09:42.579-0400 I NETWORK  [conn26] received client metadata from 127.0.0.1:60607 conn26: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:09:42.580-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60608 #27 (3 connections now open)
2018-07-18T15:09:42.580-0400 I NETWORK  [conn27] received client metadata from 127.0.0.1:60608 conn27: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:09:42.580-0400 I STORAGE  [conn27] createCollection: adstxt.posts with generated UUID: bc6bdf7e-5bd7-46d4-b530-b2eddb1ff1ef
2018-07-18T15:09:44.264-0400 I NETWORK  [conn27] end connection 127.0.0.1:60608 (2 connections now open)
2018-07-18T15:09:44.264-0400 I NETWORK  [conn26] end connection 127.0.0.1:60607 (1 connection now open)
2018-07-18T15:10:38.053-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T15:10:38.053-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T15:10:38.084-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T15:10:47.335-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60623 #28 (2 connections now open)
2018-07-18T15:10:47.335-0400 I NETWORK  [conn28] received client metadata from 127.0.0.1:60623 conn28: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:10:47.336-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60624 #29 (3 connections now open)
2018-07-18T15:10:47.337-0400 I NETWORK  [conn29] received client metadata from 127.0.0.1:60624 conn29: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:10:47.337-0400 I STORAGE  [conn29] createCollection: adstxt.posts with generated UUID: 5a3d7e91-a66f-47fb-bc7a-45e0a6f6a696
2018-07-18T15:10:48.912-0400 I NETWORK  [conn29] end connection 127.0.0.1:60624 (2 connections now open)
2018-07-18T15:10:48.912-0400 I NETWORK  [conn28] end connection 127.0.0.1:60623 (1 connection now open)
2018-07-18T15:11:33.186-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T15:11:33.186-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T15:11:33.215-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T15:11:37.292-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60636 #30 (2 connections now open)
2018-07-18T15:11:37.293-0400 I NETWORK  [conn30] received client metadata from 127.0.0.1:60636 conn30: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:11:37.800-0400 I NETWORK  [conn30] end connection 127.0.0.1:60636 (1 connection now open)
MongoDB is already running.
2018-07-18T15:11:59.018-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60646 #31 (2 connections now open)
2018-07-18T15:11:59.018-0400 I NETWORK  [conn31] received client metadata from 127.0.0.1:60646 conn31: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:11:59.019-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60647 #32 (3 connections now open)
2018-07-18T15:11:59.019-0400 I NETWORK  [conn32] received client metadata from 127.0.0.1:60647 conn32: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:11:59.020-0400 I STORAGE  [conn32] createCollection: adstxt.posts with generated UUID: 322e5219-f0fb-4064-8cb0-fdcae6a6afde
2018-07-18T15:12:00.562-0400 I NETWORK  [conn31] end connection 127.0.0.1:60646 (1 connection now open)
2018-07-18T15:12:00.562-0400 I NETWORK  [conn32] end connection 127.0.0.1:60647 (2 connections now open)
2018-07-18T15:17:33.790-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T15:17:33.790-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T15:17:33.819-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T15:17:45.776-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60695 #33 (2 connections now open)
2018-07-18T15:17:45.776-0400 I NETWORK  [conn33] received client metadata from 127.0.0.1:60695 conn33: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:17:46.289-0400 I NETWORK  [conn33] end connection 127.0.0.1:60695 (1 connection now open)
MongoDB is already running.
2018-07-18T15:18:07.064-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60705 #34 (2 connections now open)
2018-07-18T15:18:07.065-0400 I NETWORK  [conn34] received client metadata from 127.0.0.1:60705 conn34: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:18:07.066-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60706 #35 (3 connections now open)
2018-07-18T15:18:07.066-0400 I NETWORK  [conn35] received client metadata from 127.0.0.1:60706 conn35: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:18:07.066-0400 I STORAGE  [conn35] createCollection: adstxt.posts with generated UUID: 59eaf8f6-9991-41ee-9ffa-545398fc4ae9
2018-07-18T15:18:09.614-0400 I NETWORK  [conn35] end connection 127.0.0.1:60706 (2 connections now open)
2018-07-18T15:18:09.614-0400 I NETWORK  [conn34] end connection 127.0.0.1:60705 (1 connection now open)
MongoDB is already running.
2018-07-18T15:19:32.366-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60721 #36 (2 connections now open)
2018-07-18T15:19:32.366-0400 I NETWORK  [conn36] received client metadata from 127.0.0.1:60721 conn36: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:19:32.367-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60722 #37 (3 connections now open)
2018-07-18T15:19:32.367-0400 I NETWORK  [conn37] received client metadata from 127.0.0.1:60722 conn37: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:19:32.398-0400 I NETWORK  [conn37] end connection 127.0.0.1:60722 (2 connections now open)
2018-07-18T15:19:32.399-0400 I NETWORK  [conn36] end connection 127.0.0.1:60721 (1 connection now open)
MongoDB is already running.
2018-07-18T15:19:56.344-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60727 #38 (2 connections now open)
2018-07-18T15:19:56.344-0400 I NETWORK  [conn38] received client metadata from 127.0.0.1:60727 conn38: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:19:56.345-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60728 #39 (3 connections now open)
2018-07-18T15:19:56.345-0400 I NETWORK  [conn39] received client metadata from 127.0.0.1:60728 conn39: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:19:56.383-0400 I NETWORK  [conn39] end connection 127.0.0.1:60728 (2 connections now open)
2018-07-18T15:19:56.383-0400 I NETWORK  [conn38] end connection 127.0.0.1:60727 (1 connection now open)
MongoDB is already running.
2018-07-18T15:20:17.745-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60733 #40 (2 connections now open)
2018-07-18T15:20:17.746-0400 I NETWORK  [conn40] received client metadata from 127.0.0.1:60733 conn40: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:20:17.747-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60734 #41 (3 connections now open)
2018-07-18T15:20:17.747-0400 I NETWORK  [conn41] received client metadata from 127.0.0.1:60734 conn41: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:20:17.779-0400 I NETWORK  [conn41] end connection 127.0.0.1:60734 (2 connections now open)
2018-07-18T15:20:17.779-0400 I NETWORK  [conn40] end connection 127.0.0.1:60733 (1 connection now open)
MongoDB is already running.
2018-07-18T15:21:06.614-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60742 #42 (2 connections now open)
2018-07-18T15:21:06.615-0400 I NETWORK  [conn42] received client metadata from 127.0.0.1:60742 conn42: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:21:06.616-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60743 #43 (3 connections now open)
2018-07-18T15:21:06.616-0400 I NETWORK  [conn43] received client metadata from 127.0.0.1:60743 conn43: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:21:06.647-0400 I NETWORK  [conn43] end connection 127.0.0.1:60743 (2 connections now open)
2018-07-18T15:21:06.647-0400 I NETWORK  [conn42] end connection 127.0.0.1:60742 (1 connection now open)
MongoDB is already running.
2018-07-18T15:21:40.791-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60751 #44 (2 connections now open)
2018-07-18T15:21:40.791-0400 I NETWORK  [conn44] received client metadata from 127.0.0.1:60751 conn44: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:21:41.305-0400 I NETWORK  [conn44] end connection 127.0.0.1:60751 (1 connection now open)
MongoDB is already running.
2018-07-18T15:22:46.118-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60761 #45 (2 connections now open)
2018-07-18T15:22:46.119-0400 I NETWORK  [conn45] received client metadata from 127.0.0.1:60761 conn45: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:22:46.120-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60762 #46 (3 connections now open)
2018-07-18T15:22:46.120-0400 I NETWORK  [conn46] received client metadata from 127.0.0.1:60762 conn46: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:22:46.154-0400 I NETWORK  [conn46] end connection 127.0.0.1:60762 (2 connections now open)
2018-07-18T15:22:46.155-0400 I NETWORK  [conn45] end connection 127.0.0.1:60761 (1 connection now open)
MongoDB is already running.
2018-07-18T15:24:38.586-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60785 #47 (2 connections now open)
2018-07-18T15:24:38.587-0400 I NETWORK  [conn47] received client metadata from 127.0.0.1:60785 conn47: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:24:39.098-0400 I NETWORK  [conn47] end connection 127.0.0.1:60785 (1 connection now open)
MongoDB is already running.
2018-07-18T15:26:17.451-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60861 #48 (2 connections now open)
2018-07-18T15:26:17.452-0400 I NETWORK  [conn48] received client metadata from 127.0.0.1:60861 conn48: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:26:17.963-0400 I NETWORK  [conn48] end connection 127.0.0.1:60861 (1 connection now open)
MongoDB is already running.
2018-07-18T15:26:30.856-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60867 #49 (2 connections now open)
2018-07-18T15:26:30.856-0400 I NETWORK  [conn49] received client metadata from 127.0.0.1:60867 conn49: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:26:30.858-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60868 #50 (3 connections now open)
2018-07-18T15:26:30.858-0400 I NETWORK  [conn50] received client metadata from 127.0.0.1:60868 conn50: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:26:30.891-0400 I NETWORK  [conn50] end connection 127.0.0.1:60868 (2 connections now open)
2018-07-18T15:26:30.891-0400 I NETWORK  [conn49] end connection 127.0.0.1:60867 (1 connection now open)
MongoDB is already running.
2018-07-18T15:26:56.374-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60875 #51 (2 connections now open)
2018-07-18T15:26:56.374-0400 I NETWORK  [conn51] received client metadata from 127.0.0.1:60875 conn51: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:26:56.887-0400 I NETWORK  [conn51] end connection 127.0.0.1:60875 (1 connection now open)
MongoDB is already running.
2018-07-18T15:27:06.751-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60881 #52 (2 connections now open)
2018-07-18T15:27:06.751-0400 I NETWORK  [conn52] received client metadata from 127.0.0.1:60881 conn52: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:27:06.752-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60882 #53 (3 connections now open)
2018-07-18T15:27:06.753-0400 I NETWORK  [conn53] received client metadata from 127.0.0.1:60882 conn53: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:27:06.787-0400 I NETWORK  [conn53] end connection 127.0.0.1:60882 (2 connections now open)
2018-07-18T15:27:06.787-0400 I NETWORK  [conn52] end connection 127.0.0.1:60881 (1 connection now open)
MongoDB is already running.
2018-07-18T15:27:25.450-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60887 #54 (2 connections now open)
2018-07-18T15:27:25.451-0400 I NETWORK  [conn54] received client metadata from 127.0.0.1:60887 conn54: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:27:25.963-0400 I NETWORK  [conn54] end connection 127.0.0.1:60887 (1 connection now open)
MongoDB is already running.
2018-07-18T15:27:46.483-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60895 #55 (2 connections now open)
2018-07-18T15:27:46.483-0400 I NETWORK  [conn55] received client metadata from 127.0.0.1:60895 conn55: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:27:46.990-0400 I NETWORK  [conn55] end connection 127.0.0.1:60895 (1 connection now open)
MongoDB is already running.
2018-07-18T15:27:59.565-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60899 #56 (2 connections now open)
2018-07-18T15:27:59.565-0400 I NETWORK  [conn56] received client metadata from 127.0.0.1:60899 conn56: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:27:59.567-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60900 #57 (3 connections now open)
2018-07-18T15:27:59.567-0400 I NETWORK  [conn57] received client metadata from 127.0.0.1:60900 conn57: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:27:59.602-0400 I NETWORK  [conn57] end connection 127.0.0.1:60900 (2 connections now open)
2018-07-18T15:27:59.602-0400 I NETWORK  [conn56] end connection 127.0.0.1:60899 (1 connection now open)
MongoDB is already running.
2018-07-18T15:28:19.251-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60906 #58 (2 connections now open)
2018-07-18T15:28:19.251-0400 I NETWORK  [conn58] received client metadata from 127.0.0.1:60906 conn58: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:28:19.761-0400 I NETWORK  [conn58] end connection 127.0.0.1:60906 (1 connection now open)
MongoDB is already running.
2018-07-18T15:28:29.272-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60911 #59 (2 connections now open)
2018-07-18T15:28:29.272-0400 I NETWORK  [conn59] received client metadata from 127.0.0.1:60911 conn59: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:28:29.274-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60912 #60 (3 connections now open)
2018-07-18T15:28:29.274-0400 I NETWORK  [conn60] received client metadata from 127.0.0.1:60912 conn60: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:28:29.312-0400 I NETWORK  [conn60] end connection 127.0.0.1:60912 (2 connections now open)
2018-07-18T15:28:29.312-0400 I NETWORK  [conn59] end connection 127.0.0.1:60911 (1 connection now open)
MongoDB is already running.
2018-07-18T15:29:57.490-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60922 #61 (2 connections now open)
2018-07-18T15:29:57.490-0400 I NETWORK  [conn61] received client metadata from 127.0.0.1:60922 conn61: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:29:58.000-0400 I NETWORK  [conn61] end connection 127.0.0.1:60922 (1 connection now open)
MongoDB is already running.
2018-07-18T15:30:21.120-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60927 #62 (2 connections now open)
2018-07-18T15:30:21.121-0400 I NETWORK  [conn62] received client metadata from 127.0.0.1:60927 conn62: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:30:21.122-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60928 #63 (3 connections now open)
2018-07-18T15:30:21.122-0400 I NETWORK  [conn63] received client metadata from 127.0.0.1:60928 conn63: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:30:21.160-0400 I NETWORK  [conn62] end connection 127.0.0.1:60927 (1 connection now open)
2018-07-18T15:30:21.160-0400 I NETWORK  [conn63] end connection 127.0.0.1:60928 (2 connections now open)
2018-07-18T15:31:42.177-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T15:31:42.178-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T15:31:42.205-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T15:31:48.560-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60939 #64 (2 connections now open)
2018-07-18T15:31:48.560-0400 I NETWORK  [conn64] received client metadata from 127.0.0.1:60939 conn64: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:31:48.562-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60940 #65 (3 connections now open)
2018-07-18T15:31:48.563-0400 I NETWORK  [conn65] received client metadata from 127.0.0.1:60940 conn65: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:31:48.563-0400 I STORAGE  [conn65] createCollection: adstxt.posts with generated UUID: b80596d3-5630-4538-a895-b489107c33da
2018-07-18T15:31:48.638-0400 I NETWORK  [conn65] end connection 127.0.0.1:60940 (2 connections now open)
2018-07-18T15:31:48.638-0400 I NETWORK  [conn64] end connection 127.0.0.1:60939 (1 connection now open)
2018-07-18T15:32:44.168-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T15:32:44.168-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T15:32:44.193-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T15:32:50.313-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60948 #66 (2 connections now open)
2018-07-18T15:32:50.314-0400 I NETWORK  [conn66] received client metadata from 127.0.0.1:60948 conn66: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:32:50.315-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60949 #67 (3 connections now open)
2018-07-18T15:32:50.315-0400 I NETWORK  [conn67] received client metadata from 127.0.0.1:60949 conn67: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:32:50.315-0400 I STORAGE  [conn67] createCollection: adstxt.posts with generated UUID: 7ac2cb3e-56ef-4e86-a971-913675d41ad7
2018-07-18T15:32:50.400-0400 I NETWORK  [conn67] end connection 127.0.0.1:60949 (2 connections now open)
2018-07-18T15:32:50.400-0400 I NETWORK  [conn66] end connection 127.0.0.1:60948 (1 connection now open)
2018-07-18T15:36:59.479-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T15:36:59.479-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T15:36:59.505-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T15:37:05.296-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60972 #68 (2 connections now open)
2018-07-18T15:37:05.297-0400 I NETWORK  [conn68] received client metadata from 127.0.0.1:60972 conn68: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:37:05.298-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:60973 #69 (3 connections now open)
2018-07-18T15:37:05.298-0400 I NETWORK  [conn69] received client metadata from 127.0.0.1:60973 conn69: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:37:05.298-0400 I STORAGE  [conn69] createCollection: adstxt.posts with generated UUID: 888e6b98-72da-4889-8c21-71e6c3a9152a
2018-07-18T15:37:05.372-0400 I NETWORK  [conn69] end connection 127.0.0.1:60973 (2 connections now open)
2018-07-18T15:37:05.372-0400 I NETWORK  [conn68] end connection 127.0.0.1:60972 (1 connection now open)
MongoDB is already running.
2018-07-18T15:38:02.752-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61002 #70 (2 connections now open)
2018-07-18T15:38:02.753-0400 I NETWORK  [conn70] received client metadata from 127.0.0.1:61002 conn70: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:38:02.754-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61003 #71 (3 connections now open)
2018-07-18T15:38:02.754-0400 I NETWORK  [conn71] received client metadata from 127.0.0.1:61003 conn71: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:38:02.788-0400 I NETWORK  [conn71] end connection 127.0.0.1:61003 (2 connections now open)
2018-07-18T15:38:02.788-0400 I NETWORK  [conn70] end connection 127.0.0.1:61002 (1 connection now open)
MongoDB is already running.
2018-07-18T15:39:53.959-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61041 #72 (2 connections now open)
2018-07-18T15:39:53.959-0400 I NETWORK  [conn72] received client metadata from 127.0.0.1:61041 conn72: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:39:53.961-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61042 #73 (3 connections now open)
2018-07-18T15:39:53.961-0400 I NETWORK  [conn73] received client metadata from 127.0.0.1:61042 conn73: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:39:53.994-0400 I NETWORK  [conn73] end connection 127.0.0.1:61042 (2 connections now open)
2018-07-18T15:39:53.994-0400 I NETWORK  [conn72] end connection 127.0.0.1:61041 (1 connection now open)
MongoDB is already running.
2018-07-18T15:40:55.753-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61050 #74 (2 connections now open)
2018-07-18T15:40:55.753-0400 I NETWORK  [conn74] received client metadata from 127.0.0.1:61050 conn74: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:40:55.755-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61051 #75 (3 connections now open)
2018-07-18T15:40:55.755-0400 I NETWORK  [conn75] received client metadata from 127.0.0.1:61051 conn75: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:40:55.786-0400 I NETWORK  [conn75] end connection 127.0.0.1:61051 (2 connections now open)
2018-07-18T15:40:55.786-0400 I NETWORK  [conn74] end connection 127.0.0.1:61050 (1 connection now open)
MongoDB is already running.
2018-07-18T15:41:40.578-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61120 #76 (2 connections now open)
2018-07-18T15:41:40.579-0400 I NETWORK  [conn76] received client metadata from 127.0.0.1:61120 conn76: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:41:40.580-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61121 #77 (3 connections now open)
2018-07-18T15:41:40.580-0400 I NETWORK  [conn77] received client metadata from 127.0.0.1:61121 conn77: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:41:40.613-0400 I NETWORK  [conn77] end connection 127.0.0.1:61121 (2 connections now open)
2018-07-18T15:41:40.614-0400 I NETWORK  [conn76] end connection 127.0.0.1:61120 (1 connection now open)
MongoDB is already running.
2018-07-18T15:42:13.043-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61127 #78 (2 connections now open)
2018-07-18T15:42:13.044-0400 I NETWORK  [conn78] received client metadata from 127.0.0.1:61127 conn78: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:42:13.045-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61128 #79 (3 connections now open)
2018-07-18T15:42:13.045-0400 I NETWORK  [conn79] received client metadata from 127.0.0.1:61128 conn79: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:42:13.089-0400 I NETWORK  [conn79] end connection 127.0.0.1:61128 (2 connections now open)
2018-07-18T15:42:13.090-0400 I NETWORK  [conn78] end connection 127.0.0.1:61127 (1 connection now open)
2018-07-18T15:46:38.281-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T15:46:38.281-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T15:46:38.306-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T15:47:48.498-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61282 #80 (2 connections now open)
2018-07-18T15:47:48.499-0400 I NETWORK  [conn80] received client metadata from 127.0.0.1:61282 conn80: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:47:48.500-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61283 #81 (3 connections now open)
2018-07-18T15:47:48.500-0400 I NETWORK  [conn81] received client metadata from 127.0.0.1:61283 conn81: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:47:48.500-0400 I STORAGE  [conn81] createCollection: adstxt.posts with generated UUID: 7a723261-4a27-468e-8c4f-241e24e2072d
2018-07-18T15:47:48.588-0400 I NETWORK  [conn81] end connection 127.0.0.1:61283 (2 connections now open)
2018-07-18T15:47:48.590-0400 I NETWORK  [conn80] end connection 127.0.0.1:61282 (1 connection now open)
MongoDB is already running.
2018-07-18T15:48:16.413-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61288 #82 (2 connections now open)
2018-07-18T15:48:16.414-0400 I NETWORK  [conn82] received client metadata from 127.0.0.1:61288 conn82: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:48:16.415-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61289 #83 (3 connections now open)
2018-07-18T15:48:16.415-0400 I NETWORK  [conn83] received client metadata from 127.0.0.1:61289 conn83: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:48:16.450-0400 I NETWORK  [conn83] end connection 127.0.0.1:61289 (2 connections now open)
2018-07-18T15:48:16.451-0400 I NETWORK  [conn82] end connection 127.0.0.1:61288 (1 connection now open)
MongoDB is already running.
2018-07-18T15:49:25.907-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61301 #84 (2 connections now open)
2018-07-18T15:49:25.907-0400 I NETWORK  [conn84] received client metadata from 127.0.0.1:61301 conn84: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:49:25.908-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61302 #85 (3 connections now open)
2018-07-18T15:49:25.908-0400 I NETWORK  [conn85] received client metadata from 127.0.0.1:61302 conn85: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:49:25.939-0400 I NETWORK  [conn84] end connection 127.0.0.1:61301 (1 connection now open)
2018-07-18T15:49:25.939-0400 I NETWORK  [conn85] end connection 127.0.0.1:61302 (2 connections now open)
MongoDB is already running.
2018-07-18T15:50:55.312-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61311 #86 (2 connections now open)
2018-07-18T15:50:55.312-0400 I NETWORK  [conn86] received client metadata from 127.0.0.1:61311 conn86: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:50:55.314-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61312 #87 (3 connections now open)
2018-07-18T15:50:55.314-0400 I NETWORK  [conn87] received client metadata from 127.0.0.1:61312 conn87: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:50:55.340-0400 I NETWORK  [conn87] end connection 127.0.0.1:61312 (2 connections now open)
2018-07-18T15:50:55.340-0400 I NETWORK  [conn86] end connection 127.0.0.1:61311 (1 connection now open)
MongoDB is already running.
2018-07-18T15:51:21.229-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61318 #88 (2 connections now open)
2018-07-18T15:51:21.229-0400 I NETWORK  [conn88] received client metadata from 127.0.0.1:61318 conn88: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:51:21.230-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61319 #89 (3 connections now open)
2018-07-18T15:51:21.230-0400 I NETWORK  [conn89] received client metadata from 127.0.0.1:61319 conn89: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:51:21.254-0400 I NETWORK  [conn89] end connection 127.0.0.1:61319 (2 connections now open)
2018-07-18T15:51:21.254-0400 I NETWORK  [conn88] end connection 127.0.0.1:61318 (1 connection now open)
MongoDB is already running.
2018-07-18T15:51:50.527-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61325 #90 (2 connections now open)
2018-07-18T15:51:50.527-0400 I NETWORK  [conn90] received client metadata from 127.0.0.1:61325 conn90: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:51:50.528-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61326 #91 (3 connections now open)
2018-07-18T15:51:50.528-0400 I NETWORK  [conn91] received client metadata from 127.0.0.1:61326 conn91: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:51:50.559-0400 I NETWORK  [conn91] end connection 127.0.0.1:61326 (2 connections now open)
2018-07-18T15:51:50.559-0400 I NETWORK  [conn90] end connection 127.0.0.1:61325 (1 connection now open)
2018-07-18T15:52:38.067-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T15:52:38.067-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T15:52:38.092-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T15:52:48.719-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61335 #92 (2 connections now open)
2018-07-18T15:52:48.719-0400 I NETWORK  [conn92] received client metadata from 127.0.0.1:61335 conn92: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:52:48.720-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61336 #93 (3 connections now open)
2018-07-18T15:52:48.721-0400 I NETWORK  [conn93] received client metadata from 127.0.0.1:61336 conn93: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:52:48.721-0400 I STORAGE  [conn93] createCollection: adstxt.posts with generated UUID: e6e35dcf-bcf7-4ab6-b85f-47222a0f984c
2018-07-18T15:52:48.803-0400 I NETWORK  [conn92] end connection 127.0.0.1:61335 (1 connection now open)
2018-07-18T15:52:48.803-0400 I NETWORK  [conn93] end connection 127.0.0.1:61336 (2 connections now open)
MongoDB is already running.
2018-07-18T15:54:31.103-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61346 #94 (2 connections now open)
2018-07-18T15:54:31.104-0400 I NETWORK  [conn94] received client metadata from 127.0.0.1:61346 conn94: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:54:31.105-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61347 #95 (3 connections now open)
2018-07-18T15:54:31.105-0400 I NETWORK  [conn95] received client metadata from 127.0.0.1:61347 conn95: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:54:31.138-0400 I NETWORK  [conn95] end connection 127.0.0.1:61347 (2 connections now open)
2018-07-18T15:54:31.138-0400 I NETWORK  [conn94] end connection 127.0.0.1:61346 (1 connection now open)
MongoDB is already running.
2018-07-18T15:56:46.797-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61362 #96 (2 connections now open)
2018-07-18T15:56:46.798-0400 I NETWORK  [conn96] received client metadata from 127.0.0.1:61362 conn96: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:56:47.311-0400 I NETWORK  [conn96] end connection 127.0.0.1:61362 (1 connection now open)
MongoDB is already running.
2018-07-18T15:57:31.193-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61369 #97 (2 connections now open)
2018-07-18T15:57:31.193-0400 I NETWORK  [conn97] received client metadata from 127.0.0.1:61369 conn97: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:57:31.194-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61370 #98 (3 connections now open)
2018-07-18T15:57:31.194-0400 I NETWORK  [conn98] received client metadata from 127.0.0.1:61370 conn98: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:57:31.230-0400 I NETWORK  [conn98] end connection 127.0.0.1:61370 (2 connections now open)
2018-07-18T15:57:31.231-0400 I NETWORK  [conn97] end connection 127.0.0.1:61369 (1 connection now open)
2018-07-18T15:59:22.520-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T15:59:22.520-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T15:59:22.548-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T15:59:31.261-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61390 #99 (2 connections now open)
2018-07-18T15:59:31.261-0400 I NETWORK  [conn99] received client metadata from 127.0.0.1:61390 conn99: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:59:31.770-0400 I NETWORK  [conn99] end connection 127.0.0.1:61390 (1 connection now open)
MongoDB is already running.
2018-07-18T15:59:51.949-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61403 #100 (2 connections now open)
2018-07-18T15:59:51.949-0400 I NETWORK  [conn100] received client metadata from 127.0.0.1:61403 conn100: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:59:51.950-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61404 #101 (3 connections now open)
2018-07-18T15:59:51.951-0400 I NETWORK  [conn101] received client metadata from 127.0.0.1:61404 conn101: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T15:59:51.951-0400 I STORAGE  [conn101] createCollection: adstxt.posts with generated UUID: 5e30ec4f-b89f-4371-aabb-482514250d1c
2018-07-18T15:59:54.309-0400 I NETWORK  [conn101] end connection 127.0.0.1:61404 (2 connections now open)
2018-07-18T15:59:54.309-0400 I NETWORK  [conn100] end connection 127.0.0.1:61403 (1 connection now open)
2018-07-18T16:01:41.709-0400 I COMMAND  [conn3] dropDatabase adstxt - starting
2018-07-18T16:01:41.709-0400 I COMMAND  [conn3] dropDatabase adstxt - dropping 0 collections
2018-07-18T16:01:41.737-0400 I COMMAND  [conn3] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T16:02:30.251-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61436 #102 (2 connections now open)
2018-07-18T16:02:30.251-0400 I NETWORK  [conn102] received client metadata from 127.0.0.1:61436 conn102: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:02:30.252-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61437 #103 (3 connections now open)
2018-07-18T16:02:30.253-0400 I NETWORK  [conn103] received client metadata from 127.0.0.1:61437 conn103: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:02:30.253-0400 I STORAGE  [conn103] createCollection: adstxt.posts with generated UUID: d7ba61b1-f2ea-4551-9a36-cc7ecf894a88
Stop MongoDB.
2018-07-18T16:02:30.456-0400 I CONTROL  [signalProcessingThread] got signal 15 (Terminated: 15), will terminate after current cmd ends
2018-07-18T16:02:30.456-0400 I NETWORK  [signalProcessingThread] shutdown: going to close listening sockets...
2018-07-18T16:02:30.456-0400 I NETWORK  [signalProcessingThread] removing socket file: /tmp/mongodb-27017.sock
2018-07-18T16:02:30.457-0400 I FTDC     [signalProcessingThread] Shutting down full-time diagnostic data capture
2018-07-18T16:02:30.459-0400 I STORAGE  [signalProcessingThread] WiredTigerKVEngine shutting down
2018-07-18T16:02:30.726-0400 I STORAGE  [signalProcessingThread] shutdown: removing fs lock...
2018-07-18T16:02:30.728-0400 I CONTROL  [signalProcessingThread] now exiting
2018-07-18T16:02:30.728-0400 I CONTROL  [signalProcessingThread] shutting down with code:0
Start MongoDB.
2018-07-18T16:03:10.989-0400 I CONTROL  [initandlisten] MongoDB starting : pid=8348 port=27017 dbpath=/data/db 64-bit host=mm-mac-7505.local
2018-07-18T16:03:10.990-0400 I CONTROL  [initandlisten] db version v3.6.5
2018-07-18T16:03:10.990-0400 I CONTROL  [initandlisten] git version: a20ecd3e3a174162052ff99913bc2ca9a839d618
2018-07-18T16:03:10.990-0400 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018
2018-07-18T16:03:10.990-0400 I CONTROL  [initandlisten] allocator: system
2018-07-18T16:03:10.990-0400 I CONTROL  [initandlisten] modules: none
2018-07-18T16:03:10.990-0400 I CONTROL  [initandlisten] build environment:
2018-07-18T16:03:10.990-0400 I CONTROL  [initandlisten]     distarch: x86_64
2018-07-18T16:03:10.990-0400 I CONTROL  [initandlisten]     target_arch: x86_64
2018-07-18T16:03:10.990-0400 I CONTROL  [initandlisten] options: {}
2018-07-18T16:03:10.991-0400 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2018-07-18T16:03:10.991-0400 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7680M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),cache_cursors=false,log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2018-07-18T16:03:11.849-0400 I STORAGE  [initandlisten] WiredTiger message [1531944191:849906][8348:0x7fffdb5633c0], txn-recover: Main recovery loop: starting at 58/19866112
2018-07-18T16:03:11.946-0400 I STORAGE  [initandlisten] WiredTiger message [1531944191:946640][8348:0x7fffdb5633c0], txn-recover: Recovering log 58 through 59
2018-07-18T16:03:12.013-0400 I STORAGE  [initandlisten] WiredTiger message [1531944192:13326][8348:0x7fffdb5633c0], txn-recover: Recovering log 59 through 59
2018-07-18T16:03:12.070-0400 I STORAGE  [initandlisten] WiredTiger message [1531944192:70948][8348:0x7fffdb5633c0], txn-recover: Set global recovery timestamp: 0
2018-07-18T16:03:12.210-0400 I CONTROL  [initandlisten] 
2018-07-18T16:03:12.210-0400 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2018-07-18T16:03:12.210-0400 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2018-07-18T16:03:12.210-0400 I CONTROL  [initandlisten] 
2018-07-18T16:03:12.210-0400 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2018-07-18T16:03:12.210-0400 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2018-07-18T16:03:12.210-0400 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2018-07-18T16:03:12.211-0400 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2018-07-18T16:03:12.211-0400 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2018-07-18T16:03:12.211-0400 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2018-07-18T16:03:12.211-0400 I CONTROL  [initandlisten] 
2018-07-18T16:03:12.211-0400 I CONTROL  [initandlisten] 
2018-07-18T16:03:12.211-0400 I CONTROL  [initandlisten] ** WARNING: soft rlimits too low. Number of files is 256, should be at least 1000
2018-07-18T16:03:12.231-0400 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2018-07-18T16:03:12.232-0400 I NETWORK  [initandlisten] waiting for connections on port 27017
2018-07-18T16:03:12.556-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61450 #1 (1 connection now open)
2018-07-18T16:03:12.557-0400 I NETWORK  [conn1] received client metadata from 127.0.0.1:61450 conn1: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:03:12.559-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61451 #2 (2 connections now open)
2018-07-18T16:03:12.559-0400 I NETWORK  [conn2] received client metadata from 127.0.0.1:61451 conn2: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:03:14.827-0400 I NETWORK  [conn1] end connection 127.0.0.1:61450 (0 connections now open)
2018-07-18T16:03:14.827-0400 I NETWORK  [conn2] end connection 127.0.0.1:61451 (1 connection now open)
MongoDB is already running.
2018-07-18T16:03:26.085-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61461 #3 (1 connection now open)
2018-07-18T16:03:26.086-0400 I NETWORK  [conn3] received client metadata from 127.0.0.1:61461 conn3: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:03:26.087-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61462 #4 (2 connections now open)
2018-07-18T16:03:26.087-0400 I NETWORK  [conn4] received client metadata from 127.0.0.1:61462 conn4: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:03:27.768-0400 I NETWORK  [conn4] end connection 127.0.0.1:61462 (1 connection now open)
2018-07-18T16:03:27.768-0400 I NETWORK  [conn3] end connection 127.0.0.1:61461 (0 connections now open)
MongoDB is already running.
2018-07-18T16:04:13.400-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61474 #5 (1 connection now open)
2018-07-18T16:04:13.400-0400 I NETWORK  [conn5] received client metadata from 127.0.0.1:61474 conn5: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:04:13.401-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61475 #6 (2 connections now open)
2018-07-18T16:04:13.401-0400 I NETWORK  [conn6] received client metadata from 127.0.0.1:61475 conn6: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:04:14.931-0400 I NETWORK  [conn6] end connection 127.0.0.1:61475 (1 connection now open)
2018-07-18T16:04:14.931-0400 I NETWORK  [conn5] end connection 127.0.0.1:61474 (0 connections now open)
2018-07-18T16:04:19.012-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61476 #7 (1 connection now open)
2018-07-18T16:04:19.013-0400 I NETWORK  [conn7] received client metadata from 127.0.0.1:61476 conn7: { application: { name: "MongoDB Shell" }, driver: { name: "MongoDB Internal Client", version: "3.6.5" }, os: { type: "Darwin", name: "Mac OS X", architecture: "x86_64", version: "16.7.0" } }
2018-07-18T16:14:55.655-0400 I COMMAND  [conn7] dropDatabase adstxt - starting
2018-07-18T16:14:55.655-0400 I COMMAND  [conn7] dropDatabase adstxt - dropping 0 collections
2018-07-18T16:14:55.683-0400 I COMMAND  [conn7] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T16:16:11.507-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61720 #8 (2 connections now open)
2018-07-18T16:16:11.507-0400 I NETWORK  [conn8] received client metadata from 127.0.0.1:61720 conn8: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:16:11.508-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61721 #9 (3 connections now open)
2018-07-18T16:16:11.508-0400 I NETWORK  [conn9] received client metadata from 127.0.0.1:61721 conn9: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:16:11.513-0400 I STORAGE  [conn9] createCollection: adstxt.posts with generated UUID: bae777a9-9446-479a-a908-35fce0b3fd2f
2018-07-18T16:16:12.929-0400 I NETWORK  [conn9] end connection 127.0.0.1:61721 (2 connections now open)
2018-07-18T16:16:12.929-0400 I NETWORK  [conn8] end connection 127.0.0.1:61720 (1 connection now open)
MongoDB is already running.
2018-07-18T16:17:15.763-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61733 #10 (2 connections now open)
2018-07-18T16:17:15.763-0400 I NETWORK  [conn10] received client metadata from 127.0.0.1:61733 conn10: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:17:15.764-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61734 #11 (3 connections now open)
2018-07-18T16:17:15.764-0400 I NETWORK  [conn11] received client metadata from 127.0.0.1:61734 conn11: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:17:17.200-0400 I NETWORK  [conn11] end connection 127.0.0.1:61734 (2 connections now open)
2018-07-18T16:17:17.200-0400 I NETWORK  [conn10] end connection 127.0.0.1:61733 (1 connection now open)
2018-07-18T16:24:08.494-0400 I COMMAND  [conn7] dropDatabase adstxt - starting
2018-07-18T16:24:08.494-0400 I COMMAND  [conn7] dropDatabase adstxt - dropping 0 collections
2018-07-18T16:24:08.522-0400 I COMMAND  [conn7] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T16:24:15.372-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61831 #12 (2 connections now open)
2018-07-18T16:24:15.373-0400 I NETWORK  [conn12] received client metadata from 127.0.0.1:61831 conn12: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:24:15.374-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61832 #13 (3 connections now open)
2018-07-18T16:24:15.374-0400 I NETWORK  [conn13] received client metadata from 127.0.0.1:61832 conn13: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:24:15.374-0400 I STORAGE  [conn13] createCollection: adstxt.posts with generated UUID: db2d91d4-c593-4f4b-9714-785326b38ef2
2018-07-18T16:24:15.885-0400 I NETWORK  [conn12] end connection 127.0.0.1:61831 (2 connections now open)
2018-07-18T16:24:15.885-0400 I NETWORK  [conn13] end connection 127.0.0.1:61832 (1 connection now open)
MongoDB is already running.
2018-07-18T16:25:03.329-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61846 #14 (2 connections now open)
2018-07-18T16:25:03.330-0400 I NETWORK  [conn14] received client metadata from 127.0.0.1:61846 conn14: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:25:03.331-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61847 #15 (3 connections now open)
2018-07-18T16:25:03.331-0400 I NETWORK  [conn15] received client metadata from 127.0.0.1:61847 conn15: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:25:03.839-0400 I NETWORK  [conn14] end connection 127.0.0.1:61846 (1 connection now open)
2018-07-18T16:25:03.839-0400 I NETWORK  [conn15] end connection 127.0.0.1:61847 (2 connections now open)
MongoDB is already running.
2018-07-18T16:25:23.578-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61857 #16 (2 connections now open)
2018-07-18T16:25:23.579-0400 I NETWORK  [conn16] received client metadata from 127.0.0.1:61857 conn16: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:25:23.580-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61858 #17 (3 connections now open)
2018-07-18T16:25:23.580-0400 I NETWORK  [conn17] received client metadata from 127.0.0.1:61858 conn17: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:25:24.091-0400 I NETWORK  [conn16] end connection 127.0.0.1:61857 (2 connections now open)
2018-07-18T16:25:24.091-0400 I NETWORK  [conn17] end connection 127.0.0.1:61858 (1 connection now open)
MongoDB is already running.
2018-07-18T16:25:38.718-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61867 #18 (2 connections now open)
2018-07-18T16:25:38.718-0400 I NETWORK  [conn18] received client metadata from 127.0.0.1:61867 conn18: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:25:38.719-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61868 #19 (3 connections now open)
2018-07-18T16:25:38.719-0400 I NETWORK  [conn19] received client metadata from 127.0.0.1:61868 conn19: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:25:40.238-0400 I NETWORK  [conn19] end connection 127.0.0.1:61868 (2 connections now open)
2018-07-18T16:25:40.238-0400 I NETWORK  [conn18] end connection 127.0.0.1:61867 (1 connection now open)
MongoDB is already running.
2018-07-18T16:26:20.325-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61883 #20 (2 connections now open)
2018-07-18T16:26:20.325-0400 I NETWORK  [conn20] received client metadata from 127.0.0.1:61883 conn20: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:26:20.326-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61884 #21 (3 connections now open)
2018-07-18T16:26:20.326-0400 I NETWORK  [conn21] received client metadata from 127.0.0.1:61884 conn21: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:26:22.031-0400 I NETWORK  [conn20] end connection 127.0.0.1:61883 (1 connection now open)
2018-07-18T16:26:22.031-0400 I NETWORK  [conn21] end connection 127.0.0.1:61884 (2 connections now open)
MongoDB is already running.
2018-07-18T16:26:37.694-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61896 #22 (2 connections now open)
2018-07-18T16:26:37.695-0400 I NETWORK  [conn22] received client metadata from 127.0.0.1:61896 conn22: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:26:37.696-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61897 #23 (3 connections now open)
2018-07-18T16:26:37.696-0400 I NETWORK  [conn23] received client metadata from 127.0.0.1:61897 conn23: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:26:39.286-0400 I NETWORK  [conn23] end connection 127.0.0.1:61897 (2 connections now open)
2018-07-18T16:26:39.286-0400 I NETWORK  [conn22] end connection 127.0.0.1:61896 (1 connection now open)
MongoDB is already running.
2018-07-18T16:28:26.305-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61916 #24 (2 connections now open)
2018-07-18T16:28:26.306-0400 I NETWORK  [conn24] received client metadata from 127.0.0.1:61916 conn24: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:28:26.307-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:61917 #25 (3 connections now open)
2018-07-18T16:28:26.307-0400 I NETWORK  [conn25] received client metadata from 127.0.0.1:61917 conn25: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:28:27.910-0400 I NETWORK  [conn24] end connection 127.0.0.1:61916 (1 connection now open)
2018-07-18T16:28:27.910-0400 I NETWORK  [conn25] end connection 127.0.0.1:61917 (2 connections now open)
MongoDB is already running.
2018-07-18T16:31:08.640-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62008 #26 (2 connections now open)
2018-07-18T16:31:08.640-0400 I NETWORK  [conn26] received client metadata from 127.0.0.1:62008 conn26: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:31:08.642-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62009 #27 (3 connections now open)
2018-07-18T16:31:08.642-0400 I NETWORK  [conn27] received client metadata from 127.0.0.1:62009 conn27: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:31:09.153-0400 I NETWORK  [conn27] end connection 127.0.0.1:62009 (2 connections now open)
2018-07-18T16:31:09.153-0400 I NETWORK  [conn26] end connection 127.0.0.1:62008 (1 connection now open)
MongoDB is already running.
2018-07-18T16:31:42.659-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62021 #28 (2 connections now open)
2018-07-18T16:31:42.659-0400 I NETWORK  [conn28] received client metadata from 127.0.0.1:62021 conn28: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:31:42.661-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62022 #29 (3 connections now open)
2018-07-18T16:31:42.661-0400 I NETWORK  [conn29] received client metadata from 127.0.0.1:62022 conn29: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:31:44.142-0400 I NETWORK  [conn29] end connection 127.0.0.1:62022 (2 connections now open)
2018-07-18T16:31:44.142-0400 I NETWORK  [conn28] end connection 127.0.0.1:62021 (1 connection now open)
MongoDB is already running.
2018-07-18T16:32:10.269-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62033 #30 (2 connections now open)
2018-07-18T16:32:10.269-0400 I NETWORK  [conn30] received client metadata from 127.0.0.1:62033 conn30: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:32:10.271-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62034 #31 (3 connections now open)
2018-07-18T16:32:10.271-0400 I NETWORK  [conn31] received client metadata from 127.0.0.1:62034 conn31: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:32:11.906-0400 I NETWORK  [conn31] end connection 127.0.0.1:62034 (2 connections now open)
2018-07-18T16:32:11.906-0400 I NETWORK  [conn30] end connection 127.0.0.1:62033 (1 connection now open)
MongoDB is already running.
2018-07-18T16:32:23.186-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62044 #32 (2 connections now open)
2018-07-18T16:32:23.186-0400 I NETWORK  [conn32] received client metadata from 127.0.0.1:62044 conn32: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:32:23.187-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62045 #33 (3 connections now open)
2018-07-18T16:32:23.187-0400 I NETWORK  [conn33] received client metadata from 127.0.0.1:62045 conn33: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:32:23.695-0400 I NETWORK  [conn33] end connection 127.0.0.1:62045 (1 connection now open)
2018-07-18T16:32:23.695-0400 I NETWORK  [conn32] end connection 127.0.0.1:62044 (2 connections now open)
MongoDB is already running.
2018-07-18T16:32:59.040-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62055 #34 (2 connections now open)
2018-07-18T16:32:59.040-0400 I NETWORK  [conn34] received client metadata from 127.0.0.1:62055 conn34: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:32:59.041-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62056 #35 (3 connections now open)
2018-07-18T16:32:59.041-0400 I NETWORK  [conn35] received client metadata from 127.0.0.1:62056 conn35: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:33:00.663-0400 I NETWORK  [conn35] end connection 127.0.0.1:62056 (2 connections now open)
2018-07-18T16:33:00.663-0400 I NETWORK  [conn34] end connection 127.0.0.1:62055 (1 connection now open)
MongoDB is already running.
2018-07-18T16:33:20.456-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62067 #36 (2 connections now open)
2018-07-18T16:33:20.457-0400 I NETWORK  [conn36] received client metadata from 127.0.0.1:62067 conn36: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:33:20.458-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62068 #37 (3 connections now open)
2018-07-18T16:33:20.458-0400 I NETWORK  [conn37] received client metadata from 127.0.0.1:62068 conn37: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:33:21.991-0400 I NETWORK  [conn37] end connection 127.0.0.1:62068 (2 connections now open)
2018-07-18T16:33:21.991-0400 I NETWORK  [conn36] end connection 127.0.0.1:62067 (1 connection now open)
MongoDB is already running.
2018-07-18T16:34:13.863-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62080 #38 (2 connections now open)
2018-07-18T16:34:13.863-0400 I NETWORK  [conn38] received client metadata from 127.0.0.1:62080 conn38: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:34:13.864-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62081 #39 (3 connections now open)
2018-07-18T16:34:13.864-0400 I NETWORK  [conn39] received client metadata from 127.0.0.1:62081 conn39: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:34:19.879-0400 I NETWORK  [conn39] end connection 127.0.0.1:62081 (2 connections now open)
2018-07-18T16:34:19.879-0400 I NETWORK  [conn38] end connection 127.0.0.1:62080 (1 connection now open)
MongoDB is already running.
2018-07-18T16:36:27.211-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62138 #40 (2 connections now open)
2018-07-18T16:36:27.211-0400 I NETWORK  [conn40] received client metadata from 127.0.0.1:62138 conn40: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:36:27.212-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62139 #41 (3 connections now open)
2018-07-18T16:36:27.213-0400 I NETWORK  [conn41] received client metadata from 127.0.0.1:62139 conn41: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:37:42.590-0400 I NETWORK  [conn41] end connection 127.0.0.1:62139 (2 connections now open)
2018-07-18T16:37:42.590-0400 I NETWORK  [conn40] end connection 127.0.0.1:62138 (1 connection now open)
MongoDB is already running.
2018-07-18T16:41:59.559-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62239 #42 (2 connections now open)
2018-07-18T16:41:59.559-0400 I NETWORK  [conn42] received client metadata from 127.0.0.1:62239 conn42: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:41:59.560-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62240 #43 (3 connections now open)
2018-07-18T16:41:59.560-0400 I NETWORK  [conn43] received client metadata from 127.0.0.1:62240 conn43: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:42:00.068-0400 I NETWORK  [conn42] end connection 127.0.0.1:62239 (2 connections now open)
2018-07-18T16:42:00.068-0400 I NETWORK  [conn43] end connection 127.0.0.1:62240 (1 connection now open)
MongoDB is already running.
2018-07-18T16:42:17.271-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62251 #44 (2 connections now open)
2018-07-18T16:42:17.271-0400 I NETWORK  [conn44] received client metadata from 127.0.0.1:62251 conn44: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:42:17.273-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62252 #45 (3 connections now open)
2018-07-18T16:42:17.273-0400 I NETWORK  [conn45] received client metadata from 127.0.0.1:62252 conn45: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
MongoDB is already running.
2018-07-18T16:42:39.540-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62262 #46 (4 connections now open)
2018-07-18T16:42:39.540-0400 I NETWORK  [conn46] received client metadata from 127.0.0.1:62262 conn46: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:42:39.542-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62263 #47 (5 connections now open)
2018-07-18T16:42:39.542-0400 I NETWORK  [conn47] received client metadata from 127.0.0.1:62263 conn47: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:42:41.009-0400 I NETWORK  [conn47] end connection 127.0.0.1:62263 (4 connections now open)
2018-07-18T16:42:41.009-0400 I NETWORK  [conn46] end connection 127.0.0.1:62262 (3 connections now open)
2018-07-18T16:45:38.578-0400 I COMMAND  [conn7] dropDatabase adstxt - starting
2018-07-18T16:45:38.578-0400 I COMMAND  [conn7] dropDatabase adstxt - dropping 0 collections
2018-07-18T16:45:38.605-0400 I COMMAND  [conn7] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T16:45:42.356-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62348 #48 (4 connections now open)
2018-07-18T16:45:42.356-0400 I NETWORK  [conn48] received client metadata from 127.0.0.1:62348 conn48: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:45:42.357-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62349 #49 (5 connections now open)
2018-07-18T16:45:42.357-0400 I NETWORK  [conn49] received client metadata from 127.0.0.1:62349 conn49: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:45:42.358-0400 I STORAGE  [conn49] createCollection: adstxt.posts with generated UUID: 81b461aa-33f8-400b-b1f7-1e02b18ad7b9
2018-07-18T16:45:42.473-0400 I NETWORK  [conn48] end connection 127.0.0.1:62348 (3 connections now open)
2018-07-18T16:45:42.473-0400 I NETWORK  [conn49] end connection 127.0.0.1:62349 (4 connections now open)
MongoDB is already running.
2018-07-18T16:46:55.152-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62464 #50 (4 connections now open)
2018-07-18T16:46:55.152-0400 I NETWORK  [conn50] received client metadata from 127.0.0.1:62464 conn50: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:46:55.153-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62465 #51 (5 connections now open)
2018-07-18T16:46:55.153-0400 I NETWORK  [conn51] received client metadata from 127.0.0.1:62465 conn51: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:46:55.208-0400 I NETWORK  [conn50] end connection 127.0.0.1:62464 (3 connections now open)
2018-07-18T16:46:55.208-0400 I NETWORK  [conn51] end connection 127.0.0.1:62465 (4 connections now open)
2018-07-18T16:49:20.300-0400 I COMMAND  [conn7] dropDatabase adstxt - starting
2018-07-18T16:49:20.300-0400 I COMMAND  [conn7] dropDatabase adstxt - dropping 0 collections
2018-07-18T16:49:20.325-0400 I COMMAND  [conn7] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T16:49:27.079-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62486 #52 (4 connections now open)
2018-07-18T16:49:27.079-0400 I NETWORK  [conn52] received client metadata from 127.0.0.1:62486 conn52: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:49:27.081-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62487 #53 (5 connections now open)
2018-07-18T16:49:27.081-0400 I NETWORK  [conn53] received client metadata from 127.0.0.1:62487 conn53: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:49:27.081-0400 I STORAGE  [conn53] createCollection: adstxt.posts with generated UUID: 377b3be4-10cd-479e-a650-bcbb569a9c38
2018-07-18T16:49:27.191-0400 I NETWORK  [conn53] end connection 127.0.0.1:62487 (4 connections now open)
2018-07-18T16:49:27.191-0400 I NETWORK  [conn52] end connection 127.0.0.1:62486 (3 connections now open)
MongoDB is already running.
2018-07-18T16:50:49.443-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62544 #54 (4 connections now open)
2018-07-18T16:50:49.443-0400 I NETWORK  [conn54] received client metadata from 127.0.0.1:62544 conn54: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:50:49.445-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62545 #55 (5 connections now open)
2018-07-18T16:50:49.445-0400 I NETWORK  [conn55] received client metadata from 127.0.0.1:62545 conn55: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:50:49.502-0400 I NETWORK  [conn54] end connection 127.0.0.1:62544 (4 connections now open)
2018-07-18T16:50:49.502-0400 I NETWORK  [conn55] end connection 127.0.0.1:62545 (3 connections now open)
2018-07-18T16:53:16.409-0400 I COMMAND  [conn7] dropDatabase adstxt - starting
2018-07-18T16:53:16.409-0400 I COMMAND  [conn7] dropDatabase adstxt - dropping 0 collections
2018-07-18T16:53:16.434-0400 I COMMAND  [conn7] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T16:53:22.889-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62565 #56 (4 connections now open)
2018-07-18T16:53:22.889-0400 I NETWORK  [conn56] received client metadata from 127.0.0.1:62565 conn56: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:53:22.890-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62566 #57 (5 connections now open)
2018-07-18T16:53:22.890-0400 I NETWORK  [conn57] received client metadata from 127.0.0.1:62566 conn57: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:53:22.891-0400 I STORAGE  [conn57] createCollection: adstxt.posts with generated UUID: b3087da7-f356-43ca-98e9-9facc60de613
2018-07-18T16:53:23.399-0400 I NETWORK  [conn56] end connection 127.0.0.1:62565 (4 connections now open)
2018-07-18T16:53:23.399-0400 I NETWORK  [conn57] end connection 127.0.0.1:62566 (3 connections now open)
MongoDB is already running.
2018-07-18T16:53:36.421-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62576 #58 (4 connections now open)
2018-07-18T16:53:36.421-0400 I NETWORK  [conn58] received client metadata from 127.0.0.1:62576 conn58: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:53:36.422-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62577 #59 (5 connections now open)
2018-07-18T16:53:36.423-0400 I NETWORK  [conn59] received client metadata from 127.0.0.1:62577 conn59: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:53:36.932-0400 I NETWORK  [conn59] end connection 127.0.0.1:62577 (4 connections now open)
2018-07-18T16:53:36.932-0400 I NETWORK  [conn58] end connection 127.0.0.1:62576 (3 connections now open)
MongoDB is already running.
2018-07-18T16:54:03.265-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62588 #60 (4 connections now open)
2018-07-18T16:54:03.266-0400 I NETWORK  [conn60] received client metadata from 127.0.0.1:62588 conn60: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:54:03.267-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62589 #61 (5 connections now open)
2018-07-18T16:54:03.267-0400 I NETWORK  [conn61] received client metadata from 127.0.0.1:62589 conn61: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:54:03.319-0400 I NETWORK  [conn61] end connection 127.0.0.1:62589 (4 connections now open)
2018-07-18T16:54:03.319-0400 I NETWORK  [conn60] end connection 127.0.0.1:62588 (3 connections now open)
MongoDB is already running.
2018-07-18T16:54:22.664-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62599 #62 (4 connections now open)
2018-07-18T16:54:22.664-0400 I NETWORK  [conn62] received client metadata from 127.0.0.1:62599 conn62: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:54:22.666-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62600 #63 (5 connections now open)
2018-07-18T16:54:22.666-0400 I NETWORK  [conn63] received client metadata from 127.0.0.1:62600 conn63: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:54:22.720-0400 I NETWORK  [conn62] end connection 127.0.0.1:62599 (3 connections now open)
2018-07-18T16:54:22.720-0400 I NETWORK  [conn63] end connection 127.0.0.1:62600 (4 connections now open)
MongoDB is already running.
2018-07-18T16:54:36.965-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62609 #64 (4 connections now open)
2018-07-18T16:54:36.965-0400 I NETWORK  [conn64] received client metadata from 127.0.0.1:62609 conn64: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:54:36.966-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62610 #65 (5 connections now open)
2018-07-18T16:54:36.967-0400 I NETWORK  [conn65] received client metadata from 127.0.0.1:62610 conn65: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T16:54:37.473-0400 I NETWORK  [conn65] end connection 127.0.0.1:62610 (4 connections now open)
2018-07-18T16:54:37.473-0400 I NETWORK  [conn64] end connection 127.0.0.1:62609 (3 connections now open)
MongoDB is already running.
2018-07-18T17:01:05.270-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62809 #66 (4 connections now open)
2018-07-18T17:01:05.271-0400 I NETWORK  [conn66] received client metadata from 127.0.0.1:62809 conn66: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:01:05.272-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62810 #67 (5 connections now open)
2018-07-18T17:01:05.272-0400 I NETWORK  [conn67] received client metadata from 127.0.0.1:62810 conn67: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:01:05.780-0400 I NETWORK  [conn66] end connection 127.0.0.1:62809 (4 connections now open)
2018-07-18T17:01:05.780-0400 I NETWORK  [conn67] end connection 127.0.0.1:62810 (3 connections now open)
MongoDB is already running.
2018-07-18T17:01:37.804-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62821 #68 (4 connections now open)
2018-07-18T17:01:37.804-0400 I NETWORK  [conn68] received client metadata from 127.0.0.1:62821 conn68: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:01:37.805-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62822 #69 (5 connections now open)
2018-07-18T17:01:37.806-0400 I NETWORK  [conn69] received client metadata from 127.0.0.1:62822 conn69: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:01:38.318-0400 I NETWORK  [conn68] end connection 127.0.0.1:62821 (4 connections now open)
2018-07-18T17:01:38.318-0400 I NETWORK  [conn69] end connection 127.0.0.1:62822 (3 connections now open)
MongoDB is already running.
2018-07-18T17:04:59.270-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62929 #70 (4 connections now open)
2018-07-18T17:04:59.270-0400 I NETWORK  [conn70] received client metadata from 127.0.0.1:62929 conn70: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:04:59.272-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62930 #71 (5 connections now open)
2018-07-18T17:04:59.272-0400 I NETWORK  [conn71] received client metadata from 127.0.0.1:62930 conn71: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:04:59.779-0400 I NETWORK  [conn70] end connection 127.0.0.1:62929 (3 connections now open)
2018-07-18T17:04:59.779-0400 I NETWORK  [conn71] end connection 127.0.0.1:62930 (4 connections now open)
MongoDB is already running.
2018-07-18T17:05:36.949-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62942 #72 (4 connections now open)
2018-07-18T17:05:36.949-0400 I NETWORK  [conn72] received client metadata from 127.0.0.1:62942 conn72: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:05:36.950-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62943 #73 (5 connections now open)
2018-07-18T17:05:36.951-0400 I NETWORK  [conn73] received client metadata from 127.0.0.1:62943 conn73: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:05:37.459-0400 I NETWORK  [conn72] end connection 127.0.0.1:62942 (4 connections now open)
2018-07-18T17:05:37.459-0400 I NETWORK  [conn73] end connection 127.0.0.1:62943 (3 connections now open)
MongoDB is already running.
2018-07-18T17:06:17.513-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62958 #74 (4 connections now open)
2018-07-18T17:06:17.514-0400 I NETWORK  [conn74] received client metadata from 127.0.0.1:62958 conn74: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:06:17.515-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62959 #75 (5 connections now open)
2018-07-18T17:06:17.515-0400 I NETWORK  [conn75] received client metadata from 127.0.0.1:62959 conn75: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:06:18.024-0400 I NETWORK  [conn74] end connection 127.0.0.1:62958 (3 connections now open)
2018-07-18T17:06:18.024-0400 I NETWORK  [conn75] end connection 127.0.0.1:62959 (4 connections now open)
MongoDB is already running.
2018-07-18T17:06:39.487-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62969 #76 (4 connections now open)
2018-07-18T17:06:39.487-0400 I NETWORK  [conn76] received client metadata from 127.0.0.1:62969 conn76: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:06:39.488-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:62970 #77 (5 connections now open)
2018-07-18T17:06:39.488-0400 I NETWORK  [conn77] received client metadata from 127.0.0.1:62970 conn77: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:06:39.549-0400 I NETWORK  [conn77] end connection 127.0.0.1:62970 (4 connections now open)
2018-07-18T17:06:39.549-0400 I NETWORK  [conn76] end connection 127.0.0.1:62969 (3 connections now open)
2018-07-18T17:08:50.685-0400 I COMMAND  [conn7] dropDatabase adstxt - starting
2018-07-18T17:08:50.686-0400 I COMMAND  [conn7] dropDatabase adstxt - dropping 0 collections
2018-07-18T17:08:50.713-0400 I COMMAND  [conn7] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T17:09:28.137-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63028 #78 (4 connections now open)
2018-07-18T17:09:28.137-0400 I NETWORK  [conn78] received client metadata from 127.0.0.1:63028 conn78: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:09:28.138-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63029 #79 (5 connections now open)
2018-07-18T17:09:28.138-0400 I NETWORK  [conn79] received client metadata from 127.0.0.1:63029 conn79: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:09:28.139-0400 I STORAGE  [conn79] createCollection: adstxt.posts with generated UUID: f32be3b1-2e3d-4201-8b83-f4660ddb25a5
2018-07-18T17:09:28.244-0400 I NETWORK  [conn78] end connection 127.0.0.1:63028 (4 connections now open)
2018-07-18T17:09:28.244-0400 I NETWORK  [conn79] end connection 127.0.0.1:63029 (3 connections now open)
MongoDB is already running.
2018-07-18T17:09:48.845-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63039 #80 (4 connections now open)
2018-07-18T17:09:48.845-0400 I NETWORK  [conn80] received client metadata from 127.0.0.1:63039 conn80: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:09:48.846-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63040 #81 (5 connections now open)
2018-07-18T17:09:48.846-0400 I NETWORK  [conn81] received client metadata from 127.0.0.1:63040 conn81: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:09:48.896-0400 I NETWORK  [conn81] end connection 127.0.0.1:63040 (4 connections now open)
2018-07-18T17:09:48.896-0400 I NETWORK  [conn80] end connection 127.0.0.1:63039 (3 connections now open)
2018-07-18T17:11:54.948-0400 I COMMAND  [conn7] dropDatabase adstxt - starting
2018-07-18T17:11:54.949-0400 I COMMAND  [conn7] dropDatabase adstxt - dropping 0 collections
2018-07-18T17:11:54.974-0400 I COMMAND  [conn7] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T17:12:01.760-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63064 #82 (4 connections now open)
2018-07-18T17:12:01.761-0400 I NETWORK  [conn82] received client metadata from 127.0.0.1:63064 conn82: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:12:01.762-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63065 #83 (5 connections now open)
2018-07-18T17:12:01.762-0400 I NETWORK  [conn83] received client metadata from 127.0.0.1:63065 conn83: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:12:01.762-0400 I STORAGE  [conn83] createCollection: adstxt.posts with generated UUID: 8b1821e8-1d80-4a71-8cb1-6863f460665c
2018-07-18T17:12:02.270-0400 I NETWORK  [conn82] end connection 127.0.0.1:63064 (4 connections now open)
2018-07-18T17:12:02.270-0400 I NETWORK  [conn83] end connection 127.0.0.1:63065 (3 connections now open)
MongoDB is already running.
2018-07-18T17:12:14.613-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63075 #84 (4 connections now open)
2018-07-18T17:12:14.613-0400 I NETWORK  [conn84] received client metadata from 127.0.0.1:63075 conn84: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:12:14.614-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63076 #85 (5 connections now open)
2018-07-18T17:12:14.615-0400 I NETWORK  [conn85] received client metadata from 127.0.0.1:63076 conn85: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:12:14.669-0400 I NETWORK  [conn85] end connection 127.0.0.1:63076 (4 connections now open)
2018-07-18T17:12:14.669-0400 I NETWORK  [conn84] end connection 127.0.0.1:63075 (3 connections now open)
MongoDB is already running.
2018-07-18T17:12:35.458-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63086 #86 (4 connections now open)
2018-07-18T17:12:35.458-0400 I NETWORK  [conn86] received client metadata from 127.0.0.1:63086 conn86: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:12:35.460-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63087 #87 (5 connections now open)
2018-07-18T17:12:35.460-0400 I NETWORK  [conn87] received client metadata from 127.0.0.1:63087 conn87: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:12:35.537-0400 I NETWORK  [conn87] end connection 127.0.0.1:63087 (4 connections now open)
2018-07-18T17:12:35.537-0400 I NETWORK  [conn86] end connection 127.0.0.1:63086 (3 connections now open)
MongoDB is already running.
2018-07-18T17:14:20.350-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63109 #88 (4 connections now open)
2018-07-18T17:14:20.351-0400 I NETWORK  [conn88] received client metadata from 127.0.0.1:63109 conn88: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:14:20.862-0400 I NETWORK  [conn88] end connection 127.0.0.1:63109 (3 connections now open)
MongoDB is already running.
2018-07-18T17:15:48.702-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63120 #89 (4 connections now open)
2018-07-18T17:15:48.703-0400 I NETWORK  [conn89] received client metadata from 127.0.0.1:63120 conn89: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:15:49.215-0400 I NETWORK  [conn89] end connection 127.0.0.1:63120 (3 connections now open)
MongoDB is already running.
2018-07-18T17:16:32.043-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63128 #90 (4 connections now open)
2018-07-18T17:16:32.043-0400 I NETWORK  [conn90] received client metadata from 127.0.0.1:63128 conn90: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:16:32.556-0400 I NETWORK  [conn90] end connection 127.0.0.1:63128 (3 connections now open)
MongoDB is already running.
2018-07-18T17:16:54.688-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63134 #91 (4 connections now open)
2018-07-18T17:16:54.689-0400 I NETWORK  [conn91] received client metadata from 127.0.0.1:63134 conn91: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:16:55.205-0400 I NETWORK  [conn91] end connection 127.0.0.1:63134 (3 connections now open)
MongoDB is already running.
2018-07-18T17:17:22.993-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63141 #92 (4 connections now open)
2018-07-18T17:17:22.993-0400 I NETWORK  [conn92] received client metadata from 127.0.0.1:63141 conn92: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:17:23.503-0400 I NETWORK  [conn92] end connection 127.0.0.1:63141 (3 connections now open)
MongoDB is already running.
2018-07-18T17:17:49.027-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63147 #93 (4 connections now open)
2018-07-18T17:17:49.027-0400 I NETWORK  [conn93] received client metadata from 127.0.0.1:63147 conn93: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:17:49.028-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63148 #94 (5 connections now open)
2018-07-18T17:17:49.028-0400 I NETWORK  [conn94] received client metadata from 127.0.0.1:63148 conn94: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:17:49.039-0400 I NETWORK  [conn94] end connection 127.0.0.1:63148 (4 connections now open)
2018-07-18T17:17:49.039-0400 I NETWORK  [conn93] end connection 127.0.0.1:63147 (3 connections now open)
MongoDB is already running.
2018-07-18T17:18:05.817-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63158 #95 (4 connections now open)
2018-07-18T17:18:05.818-0400 I NETWORK  [conn95] received client metadata from 127.0.0.1:63158 conn95: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:18:05.819-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63159 #96 (5 connections now open)
2018-07-18T17:18:05.819-0400 I NETWORK  [conn96] received client metadata from 127.0.0.1:63159 conn96: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:18:08.328-0400 I NETWORK  [conn96] end connection 127.0.0.1:63159 (4 connections now open)
2018-07-18T17:18:08.328-0400 I NETWORK  [conn95] end connection 127.0.0.1:63158 (3 connections now open)
MongoDB is already running.
2018-07-18T17:18:29.542-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63169 #97 (4 connections now open)
2018-07-18T17:18:29.542-0400 I NETWORK  [conn97] received client metadata from 127.0.0.1:63169 conn97: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:18:29.544-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63170 #98 (5 connections now open)
2018-07-18T17:18:29.544-0400 I NETWORK  [conn98] received client metadata from 127.0.0.1:63170 conn98: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:18:32.471-0400 I NETWORK  [conn98] end connection 127.0.0.1:63170 (3 connections now open)
2018-07-18T17:18:32.471-0400 I NETWORK  [conn97] end connection 127.0.0.1:63169 (4 connections now open)
MongoDB is already running.
2018-07-18T17:20:39.242-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63188 #99 (4 connections now open)
2018-07-18T17:20:39.242-0400 I NETWORK  [conn99] received client metadata from 127.0.0.1:63188 conn99: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:20:39.244-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63189 #100 (5 connections now open)
2018-07-18T17:20:39.244-0400 I NETWORK  [conn100] received client metadata from 127.0.0.1:63189 conn100: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:20:40.742-0400 I NETWORK  [conn100] end connection 127.0.0.1:63189 (4 connections now open)
2018-07-18T17:20:40.742-0400 I NETWORK  [conn99] end connection 127.0.0.1:63188 (3 connections now open)
MongoDB is already running.
2018-07-18T17:21:03.880-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63200 #101 (4 connections now open)
2018-07-18T17:21:03.881-0400 I NETWORK  [conn101] received client metadata from 127.0.0.1:63200 conn101: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:21:03.882-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63201 #102 (5 connections now open)
2018-07-18T17:21:03.882-0400 I NETWORK  [conn102] received client metadata from 127.0.0.1:63201 conn102: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:21:05.378-0400 I NETWORK  [conn102] end connection 127.0.0.1:63201 (4 connections now open)
2018-07-18T17:21:05.378-0400 I NETWORK  [conn101] end connection 127.0.0.1:63200 (3 connections now open)
2018-07-18T17:22:06.732-0400 I COMMAND  [conn7] dropDatabase adstxt - starting
2018-07-18T17:22:06.733-0400 I COMMAND  [conn7] dropDatabase adstxt - dropping 0 collections
2018-07-18T17:22:06.764-0400 I COMMAND  [conn7] dropDatabase adstxt - finished
MongoDB is already running.
2018-07-18T17:22:13.333-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63214 #103 (4 connections now open)
2018-07-18T17:22:13.333-0400 I NETWORK  [conn103] received client metadata from 127.0.0.1:63214 conn103: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:22:13.334-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63215 #104 (5 connections now open)
2018-07-18T17:22:13.335-0400 I NETWORK  [conn104] received client metadata from 127.0.0.1:63215 conn104: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:22:13.335-0400 I STORAGE  [conn104] createCollection: adstxt.posts with generated UUID: 16e833b2-7192-45cf-bc67-a718ba404979
2018-07-18T17:22:15.352-0400 I NETWORK  [conn104] end connection 127.0.0.1:63215 (4 connections now open)
2018-07-18T17:22:15.352-0400 I NETWORK  [conn103] end connection 127.0.0.1:63214 (3 connections now open)
MongoDB is already running.
2018-07-18T17:25:45.385-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63321 #105 (4 connections now open)
2018-07-18T17:25:45.385-0400 I NETWORK  [conn105] received client metadata from 127.0.0.1:63321 conn105: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:25:46.196-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63328 #106 (5 connections now open)
2018-07-18T17:25:46.196-0400 I NETWORK  [conn106] received client metadata from 127.0.0.1:63328 conn106: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
MongoDB is already running.
2018-07-18T17:27:04.649-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63389 #107 (6 connections now open)
2018-07-18T17:27:04.649-0400 I NETWORK  [conn107] received client metadata from 127.0.0.1:63389 conn107: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:27:07.122-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63397 #108 (7 connections now open)
2018-07-18T17:27:07.122-0400 I NETWORK  [conn108] received client metadata from 127.0.0.1:63397 conn108: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:27:08.695-0400 I COMMAND  [conn108] dropDatabase adstxt - starting
2018-07-18T17:27:08.695-0400 I COMMAND  [conn108] dropDatabase adstxt - dropping 0 collections
2018-07-18T17:27:08.718-0400 I COMMAND  [conn108] dropDatabase adstxt - finished
Stop MongoDB.
2018-07-18T17:27:08.849-0400 I CONTROL  [signalProcessingThread] got signal 15 (Terminated: 15), will terminate after current cmd ends
2018-07-18T17:27:08.849-0400 I NETWORK  [signalProcessingThread] shutdown: going to close listening sockets...
2018-07-18T17:27:08.849-0400 I NETWORK  [signalProcessingThread] removing socket file: /tmp/mongodb-27017.sock
2018-07-18T17:27:08.850-0400 I FTDC     [signalProcessingThread] Shutting down full-time diagnostic data capture
2018-07-18T17:27:08.851-0400 I STORAGE  [signalProcessingThread] WiredTigerKVEngine shutting down
2018-07-18T17:27:09.003-0400 I STORAGE  [signalProcessingThread] shutdown: removing fs lock...
2018-07-18T17:27:09.004-0400 I CONTROL  [signalProcessingThread] now exiting
2018-07-18T17:27:09.004-0400 I CONTROL  [signalProcessingThread] shutting down with code:0
Start MongoDB.
2018-07-18T17:28:56.099-0400 I CONTROL  [initandlisten] MongoDB starting : pid=10545 port=27017 dbpath=/data/db 64-bit host=mm-mac-7505.local
2018-07-18T17:28:56.100-0400 I CONTROL  [initandlisten] db version v3.6.5
2018-07-18T17:28:56.100-0400 I CONTROL  [initandlisten] git version: a20ecd3e3a174162052ff99913bc2ca9a839d618
2018-07-18T17:28:56.100-0400 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018
2018-07-18T17:28:56.100-0400 I CONTROL  [initandlisten] allocator: system
2018-07-18T17:28:56.100-0400 I CONTROL  [initandlisten] modules: none
2018-07-18T17:28:56.100-0400 I CONTROL  [initandlisten] build environment:
2018-07-18T17:28:56.100-0400 I CONTROL  [initandlisten]     distarch: x86_64
2018-07-18T17:28:56.100-0400 I CONTROL  [initandlisten]     target_arch: x86_64
2018-07-18T17:28:56.100-0400 I CONTROL  [initandlisten] options: {}
2018-07-18T17:28:56.103-0400 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2018-07-18T17:28:56.103-0400 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7680M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),cache_cursors=false,log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2018-07-18T17:28:57.014-0400 I STORAGE  [initandlisten] WiredTiger message [1531949337:14344][10545:0x7fffdb5633c0], txn-recover: Main recovery loop: starting at 59/45987840
2018-07-18T17:28:57.108-0400 I STORAGE  [initandlisten] WiredTiger message [1531949337:108188][10545:0x7fffdb5633c0], txn-recover: Recovering log 59 through 60
2018-07-18T17:28:57.162-0400 I STORAGE  [initandlisten] WiredTiger message [1531949337:162117][10545:0x7fffdb5633c0], txn-recover: Recovering log 60 through 60
2018-07-18T17:28:57.235-0400 I STORAGE  [initandlisten] WiredTiger message [1531949337:235482][10545:0x7fffdb5633c0], txn-recover: Set global recovery timestamp: 0
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] 
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] 
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] 
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] 
2018-07-18T17:28:57.369-0400 I CONTROL  [initandlisten] ** WARNING: soft rlimits too low. Number of files is 256, should be at least 1000
2018-07-18T17:28:57.409-0400 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2018-07-18T17:28:57.410-0400 I NETWORK  [initandlisten] waiting for connections on port 27017
2018-07-18T17:28:57.468-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63723 #1 (1 connection now open)
2018-07-18T17:28:57.469-0400 I NETWORK  [conn1] received client metadata from 127.0.0.1:63723 conn1: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:28:57.470-0400 I NETWORK  [listener] connection accepted from 127.0.0.1:63724 #2 (2 connections now open)
2018-07-18T17:28:57.470-0400 I NETWORK  [conn2] received client metadata from 127.0.0.1:63724 conn2: { driver: { name: "PyMongo", version: "3.6.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.6" }, platform: "CPython 3.6.5.final.0" }
2018-07-18T17:28:57.476-0400 I STORAGE  [conn2] createCollection: adstxt.posts with generated UUID: ebe921bd-b960-4fec-a2e7-9dbd65f02298
2018-07-18T17:28:59.319-0400 I COMMAND  [conn2] dropDatabase adstxt - starting
2018-07-18T17:28:59.319-0400 I COMMAND  [conn2] dropDatabase adstxt - dropping 0 collections
2018-07-18T17:28:59.340-0400 I COMMAND  [conn2] dropDatabase adstxt - finished
Stop MongoDB.
2018-07-18T17:28:59.487-0400 I CONTROL  [signalProcessingThread] got signal 15 (Terminated: 15), will terminate after current cmd ends
2018-07-18T17:28:59.487-0400 I NETWORK  [signalProcessingThread] shutdown: going to close listening sockets...
2018-07-18T17:28:59.487-0400 I NETWORK  [signalProcessingThread] removing socket file: /tmp/mongodb-27017.sock
2018-07-18T17:28:59.487-0400 I FTDC     [signalProcessingThread] Shutting down full-time diagnostic data capture
2018-07-18T17:28:59.490-0400 I STORAGE  [signalProcessingThread] WiredTigerKVEngine shutting down
2018-07-18T17:28:59.757-0400 I STORAGE  [signalProcessingThread] shutdown: removing fs lock...
2018-07-18T17:28:59.759-0400 I CONTROL  [signalProcessingThread] now exiting
2018-07-18T17:28:59.759-0400 I CONTROL  [signalProcessingThread] shutting down with code:0
